var jsonfile=require("jsonfile"),lookup_elem={},manifest_elem={},format={h1:function(e){console.log(),console.log("# "+e)},h2:function(e){console.log(),console.log("## "+e)},h3:function(e){console.log(),console.log("### "+e)},h4:function(e){console.log(),console.log("#### "+e)},textbold:function(e){console.log(),console.log(),console.log("**"+e+"**")},textlist:function(e){console.log(),console.log("- "+e)},textsublist:function(e){console.log("  * "+e)},text:function(e){console.log(),console.log(e)},url:function(e){console.log(),console.log("`"+e+"`"),console.log()},codeStart:function(){console.log(),console.log("```")},codeComment:function(e){console.log(),console.log("> "+e),console.log()},codeStop:function(){console.log("```"),console.log()},code:function(e){console.log('"'+e+'"')},jsonfy:function(e){console.log("{"+e+"}")},codeRStart:function(e){console.log(),console.log("```r")},codeMongoStart:function(e){console.log(),console.log("```shell")},codeJSStart:function(e){console.log(),console.log("```javascript")},codePyStart:function(e){console.log(),console.log("```python")},codeJSONStart:function(e){console.log(),console.log("```json")},table:function(e){console.log(e)}},lookup_keys_annot={disease:"disease code used in Oncoscape naming system",source:"data source",beta:"For a specific dataset, if the beta value is true, this dataset is currently only alive in the developmental environment and won't be included in the production version of Oncoscape application.",name:"disease name",img:"images utilized for Oncoscape application",clinical:"clinic-related collections",category:"geneset and coloring collections utilized by Oncoscape application",molecular:"molecular collections",calculated:"derived colletions for PCA (one of the Oncoscape tools) and Multidimensional scaling",edges:"derived colecitons for Markers and Patient (one of the Oncoscape tools)",type:"all possible data type for each collection"},lookup_clinical_keys_annot={events:"clinical events collection organized by patient",patient:"patient collection for each disease type",drug:"chemo or other medicine administration records",newTumor:"new tumor event records for possible patients",otherMalignancy:"other maliganancy records for possible patients",radiation:"radiation administration records",followUp:"possible follow-up records","newTumor-followUp":"possible follow-up records for the new tumor events"},manifest_keys_annot={dataset:"disease code used in Oncoscape naming system",dataType:"all possible data type for each collection",date:"processed date",source:"data source",process:"process that used to derive this collection",processName:"name of the process that used to derive this collection",parent:"the collection(s) used to derive this collection"},type_keys_annot={color:"a collection for coloring in Oncoscape application",mut:"non-synonymous mutations representated as strings in this collection",mut01:"non-synonymous mutations representated as binary values in this collection",events:"clinical events collection organized by patient",patient:"patient collection for each disease type",drug:"chemo or other medicine administration records",newTumor:"new tumor event records for possible patients",otherMalignancy:"other maliganancy records for possible patients",radiation:"radiation administration records",followUp:"possible follow-up records","newTumor-followUp":"possible follow-up records for the new tumor events",genesets:"a collection of multiple genesets with specific genes in each set",methylation:"DNA methlyation data",rna:"mRNA and microRNA expression data",protein:"protein-level and phosphoprotein level (RPPA) data",psi:"percentage spliced in (PSI, \u03a8) values in RNA splicing data",facs:"Fluorescence-activated cell sorting data",cnv:"DNA copy-number data represented as Gistic score",annotation:"annotation files for a specific data type, for instance RNA splicing",chromosome:"annoataion for chromosomes",genes:"annotation data used to annotate genes",centromere:"centromere position for each chromosome on human genome",pcaScores:"calculated PCA scores for a specific data with specific genesets",pcaLoadings:"pre-calculated PCA scores for a specific data with specific genesets",mds:"Multidimensional Scaling data for a specific data with specific genesets",edges:"derived collection to describe edges between genes and patients use for Markers and Patients (one of the Oncoscape tools)",ptDegree:"derived collection to describe the weight of patients based on on the number of data points use for Markers and Patients (one of the Oncoscape tools)",geneDegree:"derived collection to describe the weight of genes based on on the number of data points use for Markers and Patients (one of the Oncoscape tools)"},source_keys_annot={tcga:"<a href='https://gdc.cancer.gov/'>TCGA</a>",broad:"<a href='https://gdac.broadinstitute.org//'>Broad Firehose</a>",hgnc:"<a href='http://www.genenames.org/'>HUGO Gene Nomenclature Committee</a>",cBio:"<a href='http://www.cbioportal.org/'>cBioPortal</a>",bradleyLab:"<a href='www.fredhutch.org/en/labs/labs-import/bradley-lab.html/'>Bradley Lab</a>",demo:"<a href='oncoscape.sttrcancer.org/'>Demo data source</a>","ucsc-PNAS":"<a href='http://www.pnas.org/content/113/19/5394.full/'>Publication</a>",ucsc:"<a href='https://genome-cancer.ucsc.edu/'>UCSC</a>",orgHs:"<a href='https://bioconductor.org/packages/release/data/annotation/html/org.Hs.eg.db.html/'>Genome wide annotation for Human</a>"};jsonfile.readFile("lookup_elem.json",function(e,o){lookup_elem=o}),jsonfile.readFile("manifest_elem.json",function(e,o){manifest_elem=o}),jsonfile.readFile("lookup_elem.json",function(e,o){lookup_elem=o,format.h2("From Disease Perspective"),format.text("Key words to describe lookup_oncoscape_datasource collection"),format.text("Key | Annotation"),format.table("--------- | ----------- ");for(var t=Object.keys(lookup_keys_annot),a=0;a<t.length;a++)format.table(t[a]+" | "+lookup_keys_annot[t[a]]);format.h3("Disease List");var n=lookup_elem.disease.length;format.text("Disease Code | Disease Name"),format.table("--------- | ----------- ");for(var l=0;n>l;l++)format.table(lookup_elem.disease[l]+" | "+lookup_elem.name[l]);format.h3("source"),format.text("<a href='https://gdc.cancer.gov/'>TCGA</a>"),format.h3("clinical"),format.text("Key words to describe clinical collections for each disease type"),format.text("clinical collection type | Annotation"),format.table("--------- | ----------- ");for(var s=Object.keys(lookup_clinical_keys_annot),c=0;c<s.length;c++)format.table(s[c]+" | "+type_keys_annot[s[c]]);format.h3("molecular");var i=lookup_elem.molecular.source;format.text("molecular collection source | Annotation"),format.table("--------- | ----------- ");for(var r=0;r<i.length;r++)format.table(i[r]+" | "+source_keys_annot[i[r]]);var f=lookup_elem.molecular.type;for(format.text("molecular collection type | Annotation"),format.table("--------- | ----------- "),p=0;p<f.length;p++)format.table(f[p]+" | "+type_keys_annot[f[p]]);format.h3("category");var m=lookup_elem.category.source;for(format.text("category collection source | Annotation"),format.table("--------- | ----------- "),r=0;r<m.length;r++)format.table(m[r]+" | "+source_keys_annot[m[r]]);var d=lookup_elem.category.type;for(format.text("category collection type | Annotation"),format.table("--------- | ----------- "),p=0;p<d.length;p++)format.table(d[p]+" | "+type_keys_annot[d[p]]);format.h3("calculated");var u=lookup_elem.calculated.source;for(format.text("calculated collection source | Annotation"),format.table("--------- | ----------- "),r=0;r<u.length;r++)format.table(u[r]+" | "+source_keys_annot[u[r]]);var g=lookup_elem.calculated.type;for(format.text("calculated collection type | Annotation"),format.table("--------- | ----------- "),p=0;p<g.length;p++)format.table(g[p]+" | "+type_keys_annot[g[p]]);format.h3("edges"),format.text("edges names are collections of genesets: "),format.text("edges geneset | geneset information"),format.table("--------- | ----------- ");var a=0;lookup_elem.edges.name.forEach(function(e){format.table(e+" | hg19_genesets_hgnc_import["+a+"]"),a++});var h=lookup_elem.edges.source;for(format.text("edges collection source | Annotation"),format.table("--------- | ----------- "),r=0;r<h.length;r++)format.table(h[r]+" | "+source_keys_annot[h[r]])}),jsonfile.readFile("manifest_elem.json",function(e,o){manifest_elem=o,format.h2("From Collection Perspective"),format.text("Key words to describe manifest collection"),format.text("Key | Annotation"),format.table("--------- | ----------- ");for(var t=Object.keys(manifest_keys_annot),a=0;a<t.length;a++)format.table(t[a]+" | "+manifest_keys_annot[t[a]]);format.h3("Manifest Dataset"),format.text("This field is identical to lookup_oncoscape_datasource's disease field.");var t=manifest_elem.dataset;format.text("Disease Code | Disease Name"),format.table("--------- | ----------- "),t.forEach(function(e){format.table(e+" | "+lookup_elem.name[lookup_elem.disease.indexOf(e.toLowerCase())])}),format.h3("All the datatypes within Database");var n=manifest_elem.dataType;for(format.text("collection type | Annotation"),format.table("--------- | ----------- "),p=0;p<n.length;p++)format.table(n[p]+" | "+type_keys_annot[n[p]])}),format.h1("Database Description"),format.text("We use two collections to track the metadata for all the data collections within our database. 'manifest' is organized by collection. 'lookup_oncoscape_datasource' is organized by disease.");