<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>API Reference</title>

    <style>
      .highlight table td { padding: 5px; }
.highlight table pre { margin: 0; }
.highlight, .highlight .w {
  color: #f8f8f2;
  background-color: #272822;
}
.highlight .err {
  color: #151515;
  background-color: #ac4142;
}
.highlight .c, .highlight .cd, .highlight .cm, .highlight .c1, .highlight .cs {
  color: #505050;
}
.highlight .cp {
  color: #f4bf75;
}
.highlight .nt {
  color: #f4bf75;
}
.highlight .o, .highlight .ow {
  color: #d0d0d0;
}
.highlight .p, .highlight .pi {
  color: #d0d0d0;
}
.highlight .gi {
  color: #90a959;
}
.highlight .gd {
  color: #ac4142;
}
.highlight .gh {
  color: #6a9fb5;
  background-color: #151515;
  font-weight: bold;
}
.highlight .k, .highlight .kn, .highlight .kp, .highlight .kr, .highlight .kv {
  color: #aa759f;
}
.highlight .kc {
  color: #d28445;
}
.highlight .kt {
  color: #d28445;
}
.highlight .kd {
  color: #d28445;
}
.highlight .s, .highlight .sb, .highlight .sc, .highlight .sd, .highlight .s2, .highlight .sh, .highlight .sx, .highlight .s1 {
  color: #90a959;
}
.highlight .sr {
  color: #75b5aa;
}
.highlight .si {
  color: #8f5536;
}
.highlight .se {
  color: #8f5536;
}
.highlight .nn {
  color: #f4bf75;
}
.highlight .nc {
  color: #f4bf75;
}
.highlight .no {
  color: #f4bf75;
}
.highlight .na {
  color: #6a9fb5;
}
.highlight .m, .highlight .mf, .highlight .mh, .highlight .mi, .highlight .il, .highlight .mo, .highlight .mb, .highlight .mx {
  color: #90a959;
}
.highlight .ss {
  color: #90a959;
}
    </style>
    <link href="stylesheets/screen.css" rel="stylesheet" media="screen" />
    <link href="stylesheets/print.css" rel="stylesheet" media="print" />
      <script src="javascripts/all.js"></script>
  </head>

  <body class="index" data-languages="[]">
    <a href="#" id="nav-button">
      <span>
        NAV
        <img src="images/navbar.png" />
      </span>
    </a>
    <div class="tocify-wrapper">
      <img src="images/logo.png" />
        <div class="lang-selector">
        </div>
        <div class="search">
          <input type="text" class="search" id="input-search" placeholder="Search">
        </div>
        <ul class="search-results"></ul>
      <div id="toc">
      </div>
        <ul class="toc-footer">
            <li>email to contact@oncoscape.org</li>
            <li><a target="_blank" href='https://oncoscape.sttrcancer.org'>Oncoscape</a></li>
            <li><a target="_blank" href='https://github.com/FredHutch/Oncoscape'>Oncoscape Github Site</a></li>
            <li><a target="_blank" href='http://resources.sttrcancer.org/api/data-explorer/'>Oncoscape Data API Explorer</a></li>
            <li><a target="_blank" href='http://oncoscape.sttrcancer.org/documentation/'>Oncoscape Tool Documentation</a></li>
        </ul>
    </div>
    <div class="page-wrapper">
      <div class="dark-box"></div>
      <div class="content">
        <h1 id="introduction">Introduction</h1>

<p><a target="_blank" href='https://oncoscape.sttrcancer.org/#/'>Oncoscape</a> is a data visualization platform that empowers researchers to discover novel patterns and relationships between clinical and molecular data. Through a suite of interoperable tools, Oncoscape offers a unique and intuitive approach to hypothesis refinement.For more detailed information, please read <a target="_blank" href='https://github.com/FredHutch/Oncoscape'>github repo page</a>.</p>

<ul>
<li><a href='#data-content'><strong>Data Content</strong></a>

<ul>
<li>data sources</li>
<li>data types of clinical and molecular collections</li>
</ul></li>
<li><a href='#data-provenance'><strong>Data Provenance</strong></a>

<ul>
<li>Explanation of the Oncoscape data processing pipeline</li>
<li>Oncoscape interface</li>
</ul></li>
<li><a href='#data-access'><strong>Data Access</strong></a>

<ul>
<li>Endpoint for Oncoscape Data API</li>
<li>Syntax for simple query using browser</li>
<li>Interactive web application to explore the data</li>
</ul></li>
</ul>

          <h1 id="data-content">Data Content</h1>

<p>This section is dedicated to explain the raw data Oncoscape utilizes. The section <a href='#data-provenance'>Data Provenance</a> explains how the raw data has been processed to fit into our visualization model.</p>

<h2 id="clinical-data">Clinical Data</h2>

<h3 id="data-sources">Data Sources</h3>

<p>Genomic Data Commons Data Portal (GDC) from National Institutes of Health (NIH) provides the compiled annotated clinical data.</p>

<p><a target='_blank' href='https://gdc-portal.nci.nih.gov/legacy-archive/search/f?filters=%7B%22op%22:%22and%22,%22content%22:%5B%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22cases.project.program.name%22,%22value%22:%5B%22TCGA%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.data_category%22,%22value%22:%5B%22Clinical%22%5D%7D%7D,%7B%22op%22:%22in%22,%22content%22:%7B%22field%22:%22files.data_type%22,%22value%22:%5B%22Clinical%20Supplement%22,%22Clinical%20data%22%5D%7D%7D%5D%7D&pagination=%7B%22files%22:%7B%22from%22:0,%22size%22:20,%22sort%22:%22cases.project.project_id:asc%22%7D%7D'>GDC clinical data</a></p>

<h3 id="data-type">Data Type</h3>

<table><thead>
<tr>
<th>Type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>events</td>
<td>clinical events collection organized by patient</td>
</tr>
<tr>
<td>patient</td>
<td>patient collection for each disease type</td>
</tr>
<tr>
<td>drug</td>
<td>chemo or other medicine administration records</td>
</tr>
<tr>
<td>newtumor</td>
<td>new tumor event records for possible patients</td>
</tr>
<tr>
<td>othermalignancy</td>
<td>other maliganancy records for possible patients</td>
</tr>
<tr>
<td>radiation</td>
<td>radiation administration records</td>
</tr>
<tr>
<td>followup</td>
<td>possible follow-up records</td>
</tr>
<tr>
<td>newtumor-followup</td>
<td>possible follow-up records for the new tumor events</td>
</tr>
<tr>
<td>samplemap</td>
<td>sample-patient mapping collection</td>
</tr>
</tbody></table>

<h2 id="molecular-data">Molecular Data</h2>

<h3 id="data-sources">Data Sources</h3>

<p>UCSC Xena compiled annotated normalized molecular datasets of various platforms from multiple institutes. </p>

<p><a target='_blank' href='https://xenabrowser.net/datapages/?host=https://tcga.xenahubs.net'>UCSC xena hub</a></p>

<p><a target='_blank' href='https://github.com/ucscXena/ucsc-xena-server'>UCSC xena github</a></p>

<h3 id="data-type">Data Type</h3>

<table><thead>
<tr>
<th>Type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>expr</td>
<td>Expression data including mRNA and microRNA expression data and Reverse hase protein array (RPPA) data</td>
</tr>
<tr>
<td>mut</td>
<td>non-synonymous mutations representated as strings in this collection</td>
</tr>
<tr>
<td>mut01</td>
<td>non-synonymous mutations representated as binary values in this collection</td>
</tr>
<tr>
<td>meth</td>
<td>DNA methlyation data</td>
</tr>
<tr>
<td>meth_thd</td>
<td>Thresholded DNA methlyation data</td>
</tr>
<tr>
<td>cnv</td>
<td>DNA copy-number data represented as Gistic score</td>
</tr>
<tr>
<td>cnv_thd</td>
<td>Thresholded DNA copy-number data represented as Gistic score</td>
</tr>
</tbody></table>

<h3 id="schema">Schema</h3>

<table><thead>
<tr>
<th>Schema Type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>chr_sample</td>
<td>Collections of this schema have chromosomal location inhelper.formation as keys for each record, which is a list of values with samples as keys.</td>
</tr>
<tr>
<td>hugo_sample</td>
<td>Collections of this schema have chromosomal HUGO genes as keys for each record, which is a list of values with samples as keys.</td>
</tr>
<tr>
<td>sample_pos</td>
<td>Collections of this schema have samples as keys for each record, which is a list of position.</td>
</tr>
<tr>
<td>methoprobe_sample</td>
<td>Collections of this schema have methlyation probes as keys for each record, which is a list of values with samples as keys.</td>
</tr>
</tbody></table>

<h2 id="gene-sets">Gene Sets</h2>

<table><thead>
<tr>
<th>Name</th>
<th>Description</th>
<th>Genes</th>
</tr>
</thead><tbody>
<tr>
<td><a target="_blank" href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2818769/>TCGA GBM Classifiers</a></td>
<td>Gene expression-based molecular classification of GBM subtypes (Proneural, Neural, Classical, Mesenchymal)</td>
<td><a target="_blank" href='https://dev.oncoscape.sttrcancer.io/api/lookup_genesets/?q={"name":"TCGA GBM Classifiers","$fields":["genes"]}&apikey=password'>840</a></td>
</tr>
<tr>
<td>Glioma Markers</td>
<td>Genes recurrently impacted in TCGA gliomas</td>
<td><a target="_blank" href='https://dev.oncoscape.sttrcancer.io/api/lookup_genesets/?q={"name":"Glioma Markers","$fields":["genes"]}&apikey=password'>545</a></td>
</tr>
<tr>
<td><a target="_blank" href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4320046/>TCGA Pancan Mutated</a></td>
<td>Significantly mutated genes according to the TCGA PANCAN working group (syn1750331) identified by both MuSiC and MutSig</td>
<td><a target="_blank" href='https://dev.oncoscape.sttrcancer.io/api/lookup_genesets/?q={"name":"TCGA Pancan Mutated","$fields":["genes"]}&apikey=password'>73</a></td>
</tr>
<tr>
<td><a target="_blank" href=https://www.ncbi.nlm.nih.gov/pubmed/23539594/>Oncoplex Vogelstein</a></td>
<td>Combined set from the Oncoplex gene panel and driver genes described in Vogelstein, Science 2013.</td>
<td><a target="_blank" href='https://dev.oncoscape.sttrcancer.io/api/lookup_genesets/?q={"name":"Oncoplex Vogelstein","$fields":["genes"]}&apikey=password'>274</a></td>
</tr>
<tr>
<td><a target="_blank" href=http://tests.labmed.washington.edu/UW-OncoPlex/>Oncoplex</a></td>
<td>A sequencing panel that detects mutations in genes related to cancer treatment, prognosis, and diagnosis.</td>
<td><a target="_blank" href='https://dev.oncoscape.sttrcancer.io/api/lookup_genesets/?q={"name":"Oncoplex","$fields":["genes"]}&apikey=password'>263</a></td>
</tr>
<tr>
<td><a target="_blank" href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2575803/>OSCC Expression Markers</a></td>
<td>Differentially expressed probe set comparing normal oral tissue to oral squamous cell carcinoma</td>
<td><a target="_blank" href='https://dev.oncoscape.sttrcancer.io/api/lookup_genesets/?q={"name":"OSCC Expression Markers","$fields":["genes"]}&apikey=password'>109</a></td>
</tr>
</tbody></table>

          <h1 id="data-provenance">Data Provenance</h1>

<p>This section is dedicated to explain how the raw data were processed to generate new data models to fit to Oncoscape visualization tools. The section <a href='#data-content'>Data Content</a> explains the source and type of raw data.</p>

<h2 id="pipeline">Pipeline</h2>

<p><img alt="Data Processing Pipeline" src="images/datapipeline2.svg" /></p>

<h2 id="oncoscape-interface">Oncoscape Interface</h2>

<p>We use lookup as an hand-off from data-generation to data-utilization. Lookup reminds us the data are organized by diseases. And they have the subcategories: clinical, molecular. Each document of the lookup collection is to describe the organization of all the disease-related collection structure. Within each document, except for the organization of actual data collections, there is metadata section. </p>

<p><img alt="One example of lookup do3cument" src="images/lookup_doc.png" /></p>

<p>The processed raw data are stored under &lsquo;calculated&rsquo; and &#39;edges&rsquo; in each document. The &#39;calculated&rsquo; is explained in <a href='#cluster'>Cluster</a> while the &#39;edges&rsquo; is explained in <a href='#network'>Network</a>.</p>

<h3 id="cluster">Cluster</h3>

<p>The cluster collections are generated to fit into two Oncoscape tools &#39;Markers and Patients&rsquo; and &#39;PCA&rsquo;. There are two Schemas for clutster collections: Multidimensional Scaling (MDS) and Principal component analysis (PCA).</p>

<p><strong>MDS</strong></p>

<ul>
<li><p>Input Data:</p>

<ul>
<li><a href='#data-type7'>cnv_thd</a></li>
<li><a href='#data-type7'>mut01</a></li>
</ul></li>
<li><p>Mechanism:</p></li>
</ul>

<p>cnv_thd and mut01 were combined and the distance matrix were calculated best represent the similarity of individual sample in the N-dimensional space.</p>

<ul>
<li>Schema</li>
</ul>

<p><img alt="mds collection schema" src="images/mds.png" /></p>

<p><strong>PCA</strong></p>

<ul>
<li><p>Input Data:</p>

<ul>
<li><a href='#data-type7'>expr</a> including RNA and Protein</li>
<li><a href='#data-type7'>cnv</a></li>
</ul></li>
<li><p>Mechanism:</p></li>
</ul>

<p>The collection of each class will be processed respectively that distance matrices were generated to best represent the similarity of individual sample in the N-dimensional space.</p>

<ul>
<li>Schema</li>
</ul>

<p><img alt="pca scores collection schema" src="images/pcascore.png" /></p>

<p><img alt="pca loading schema" src="images/pcaloading.png" /></p>

<p><strong>Collection Organization</strong></p>

<p>The naming convention of the derived collections includes the information from method, geneset as well as data class listed below.</p>

<table><thead>
<tr>
<th>Methods</th>
<th>Genesets</th>
<th>Data Class</th>
</tr>
</thead><tbody>
<tr>
<td>PCA</td>
<td>All Genes</td>
<td>RNA</td>
</tr>
<tr>
<td>MDS</td>
<td>TCGA Pancan Mutated</td>
<td>Protein</td>
</tr>
<tr>
<td></td>
<td>Oncoplex</td>
<td>CNV/Mut01</td>
</tr>
<tr>
<td></td>
<td>Glioma Markers</td>
<td>Mut01</td>
</tr>
<tr>
<td></td>
<td>Oncoplex Vogelstein</td>
<td>CNV</td>
</tr>
<tr>
<td></td>
<td>TCGA GBM Classifiers</td>
<td></td>
</tr>
</tbody></table>

<h3 id="network">Network</h3>

<p>The network collections are generated to fit to Oncoscape tool &#39;Markers and Patients&rsquo;. There are three Schemas for network collections: edges, patient weights and gene weights.</p>

<p><strong>Edges</strong></p>

<p>Edges contain the information to define the edges connecting one patient sample with one gene. For each record the value may be integer between -2 to +2. If one gene in one patient sample has copy number variation, this information will be represented as -2 (deletion), or -1 (loss), or +1 (gain), or +2 (amplificaiton). If this gene on this patient sample doesn&rsquo;t have copy number variation but point mutation, this information will be represented as &#39;0&rsquo;. Otherwise, no record will be shown. </p>

<ul>
<li>Schema</li>
</ul>

<p><img alt="edges schema" src="images/edge.png" /></p>

<p><strong>Patient degrees</strong></p>

<p>For a certain dataset, patient degrees record the number of altered genes (copy number variation and point mutation) for each patient.</p>

<ul>
<li>Schema</li>
</ul>

<p><img alt="ptdegree schema" src="images/ptdegree.png" /></p>

<p><strong>Gene degrees</strong></p>

<p>For a certain dataset, gene degrees record the number of patients who have this gene altered (either caused by copy number varation or point mutation).</p>

<ul>
<li>Schema</li>
</ul>

<p><img alt="genedegree schema" src="images/genedegree.png" /></p>

          <h1 id="data-access">Data Access</h1>

<p>Oncoscape provides the API service based on the traditional RESTful API data structure. Data are secured with exposed by API Gateway Kong.. The privacy is managed at collection level. You can acess the public datasets through appending &lsquo;apikey=password&rsquo;.</p>

<h2 id="example-to-access-one-collection-from-browser">Example to access one collection from browser</h2>

<p><strong>HTTP Request</strong></p>

<p>Collections are accessable at the host: http://dev.oncoscape.sttrcancer.io/api/</p>

<p>The endpoint of oncoscape API is a unique URL. Every endpoint points to a unique collection.
Below lists more details of the organization of the Oncoscape Mongo Database and the collections organized by disease type.</p>

<p><code class="prettyprint">GET http://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical/?q=&amp;apikey=password</code></p>

<h2 id="query-collection-from-browser">Query Collection from Browser</h2>

<p><strong>HTTP Request</strong></p>

<p>Filter by gender and race and only show the selected fields</p>

<p><code class="prettyprint">GET http://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical/?q={&quot;gender&quot;:&quot;MALE&quot;, &quot;race&quot;:&quot;WHITE&quot;,&quot;$fields&quot;:[&quot;gender&quot;,&quot;race&quot;,&quot;patient_ID&quot;],&quot;$skip&quot;:5,&quot;$limit&quot;:2}&amp;apikey=password</code></p>

<p>only show gender, race and patient_ID</p>

<p><code class="prettyprint">&quot;$fields&quot;:[&quot;gender&quot;,&quot;race&quot;,&quot;patient_ID&quot;]</code></p>

<p>skip the first five records</p>

<p><code class="prettyprint">&quot;$skip&quot;:5</code></p>

<p>limit the final output to two records.</p>

<p><code class="prettyprint">&quot;$limit&quot;:2</code></p>

<h2 id="explore-the-oncoscape-database">Explore the Oncoscape Database</h2>

<p><strong><a target='_blank' href='http://resources.sttrcancer.org/api/data-explorer/'>Data Explorer</a> is an interactive web application to explore the clinical collections in the database.</strong></p>

      </div>
      <div class="dark-box">
          <div class="lang-selector">
          </div>
      </div>
    </div>
  </body>
</html>
