<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>API Reference</title>

    <style>
      .highlight table td { padding: 5px; }
.highlight table pre { margin: 0; }
.highlight, .highlight .w {
  color: #f8f8f2;
  background-color: #272822;
}
.highlight .err {
  color: #151515;
  background-color: #ac4142;
}
.highlight .c, .highlight .cd, .highlight .cm, .highlight .c1, .highlight .cs {
  color: #505050;
}
.highlight .cp {
  color: #f4bf75;
}
.highlight .nt {
  color: #f4bf75;
}
.highlight .o, .highlight .ow {
  color: #d0d0d0;
}
.highlight .p, .highlight .pi {
  color: #d0d0d0;
}
.highlight .gi {
  color: #90a959;
}
.highlight .gd {
  color: #ac4142;
}
.highlight .gh {
  color: #6a9fb5;
  background-color: #151515;
  font-weight: bold;
}
.highlight .k, .highlight .kn, .highlight .kp, .highlight .kr, .highlight .kv {
  color: #aa759f;
}
.highlight .kc {
  color: #d28445;
}
.highlight .kt {
  color: #d28445;
}
.highlight .kd {
  color: #d28445;
}
.highlight .s, .highlight .sb, .highlight .sc, .highlight .sd, .highlight .s2, .highlight .sh, .highlight .sx, .highlight .s1 {
  color: #90a959;
}
.highlight .sr {
  color: #75b5aa;
}
.highlight .si {
  color: #8f5536;
}
.highlight .se {
  color: #8f5536;
}
.highlight .nn {
  color: #f4bf75;
}
.highlight .nc {
  color: #f4bf75;
}
.highlight .no {
  color: #f4bf75;
}
.highlight .na {
  color: #6a9fb5;
}
.highlight .m, .highlight .mf, .highlight .mh, .highlight .mi, .highlight .il, .highlight .mo, .highlight .mb, .highlight .mx {
  color: #90a959;
}
.highlight .ss {
  color: #90a959;
}
    </style>
    <link href="stylesheets/screen.css" rel="stylesheet" media="screen" />
    <link href="stylesheets/print.css" rel="stylesheet" media="print" />
      <script src="javascripts/all.js"></script>
  </head>

  <body class="index" data-languages="[&quot;python&quot;,&quot;javascript&quot;,&quot;r&quot;]">
    <a href="#" id="nav-button">
      <span>
        NAV
        <img src="images/navbar.png" />
      </span>
    </a>
    <div class="tocify-wrapper">
      <img src="images/logo.png" />
        <div class="lang-selector">
              <a href="#" data-language-name="python">python</a>
              <a href="#" data-language-name="javascript">javascript</a>
              <a href="#" data-language-name="r">r</a>
        </div>
        <div class="search">
          <input type="text" class="search" id="input-search" placeholder="Search">
        </div>
        <ul class="search-results"></ul>
      <div id="toc">
      </div>
        <ul class="toc-footer">
            <li>email to contact@oncoscape.org</li>
            <li><a target="_blank" href='https://oncoscape.sttrcancer.org'>Oncoscape</a></li>
            <li><a target="_blank" href='https://github.com/FredHutch/Oncoscape'>Oncoscape Github Site</a></li>
            <li><a target="_blank" href='http://resources.sttrcancer.org/api/data-explorer/'>Oncoscape Data API Explorer</a></li>
            <li><a target="_blank" href='http://resources.sttrcancer.org/oncoscape'>Oncoscape Tool Documentation</a></li>
        </ul>
    </div>
    <div class="page-wrapper">
      <div class="dark-box"></div>
      <div class="content">
        <h1 id="introduction">Introduction</h1>

<p><a href="https://oncoscape.sttrcancer.org/#/">Oncoscape</a> is a web application that hosts an integrated suite of analysis tools for users to explore hypotheses related to molecular and clinical data in order to better understand cancer biology and treatment options. For more detailed information, please read <a href="https://github.com/FredHutch/Oncoscape">wiki</a>.</p>

<h1 id="authentication">Authentication</h1>

<p>Oncoscape API authentication occurs at Mongo DB level. </p>

          <h1 id="database-description">Database Description</h1>

<p>We use two collections to track the metadata for all the data collections within our database. &lsquo;manifest&rsquo; is organized by collection. &#39;lookup_oncoscape_datasource&rsquo; is organized by disease.</p>

<h2 id="from-disease-perspective">From Disease Perspective</h2>

<p>Key words to describe lookup_oncoscape_datasource collection</p>

<table><thead>
<tr>
<th>Key</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>disease</td>
<td>disease code used in Oncoscape naming system</td>
</tr>
<tr>
<td>source</td>
<td>data source</td>
</tr>
<tr>
<td>beta</td>
<td>For a specific dataset, if the beta value is true, this dataset is currently only alive in the developmental environment and won&rsquo;t be included in the production version of Oncoscape application.</td>
</tr>
<tr>
<td>name</td>
<td>disease name</td>
</tr>
<tr>
<td>img</td>
<td>images utilized for Oncoscape application</td>
</tr>
<tr>
<td>clinical</td>
<td>clinic-related collections</td>
</tr>
<tr>
<td>category</td>
<td>geneset and coloring collections utilized by Oncoscape application</td>
</tr>
<tr>
<td>molecular</td>
<td>molecular collections</td>
</tr>
<tr>
<td>calculated</td>
<td>derived colletions for PCA (one of the Oncoscape tools) and Multidimensional scaling</td>
</tr>
<tr>
<td>edges</td>
<td>derived colecitons for Markers and Patient (one of the Oncoscape tools)</td>
</tr>
<tr>
<td>type</td>
<td>all possible data type for each collection</td>
</tr>
</tbody></table>

<h3 id="disease-list">Disease List</h3>

<table><thead>
<tr>
<th>Disease Code</th>
<th>Disease Name</th>
</tr>
</thead><tbody>
<tr>
<td>brain</td>
<td>Brain</td>
</tr>
<tr>
<td>brca</td>
<td>Breast</td>
</tr>
<tr>
<td>esca</td>
<td>Esophageal</td>
</tr>
<tr>
<td>gbm</td>
<td>Glioblastoma</td>
</tr>
<tr>
<td>hnsc</td>
<td>Head and Neck</td>
</tr>
<tr>
<td>kich</td>
<td>Kidney chromophobe</td>
</tr>
<tr>
<td>kirc</td>
<td>Kidney renal clear cell</td>
</tr>
<tr>
<td>kirp</td>
<td>Kidney renal papillary cell</td>
</tr>
<tr>
<td>lgg</td>
<td>Lower grade glioma</td>
</tr>
<tr>
<td>lihc</td>
<td>Liver</td>
</tr>
<tr>
<td>luad</td>
<td>Lung adenocarcinoma</td>
</tr>
<tr>
<td>lusc</td>
<td>Lung squamous cell</td>
</tr>
<tr>
<td>sarc</td>
<td>Sarcoma</td>
</tr>
<tr>
<td>paad</td>
<td>Pancreas</td>
</tr>
<tr>
<td>prad</td>
<td>Prostate</td>
</tr>
<tr>
<td>skcm</td>
<td>Skin cutaneous melanoma</td>
</tr>
<tr>
<td>stad</td>
<td>Stomach</td>
</tr>
<tr>
<td>thca</td>
<td>Thyroid carcinoma</td>
</tr>
<tr>
<td>ucec</td>
<td>Uterine corpus endometrial</td>
</tr>
<tr>
<td>acc</td>
<td>Adrenocortical carcinoma</td>
</tr>
<tr>
<td>blca</td>
<td>Bladder urothelial carcinoma</td>
</tr>
<tr>
<td>cesc</td>
<td>Cervical</td>
</tr>
<tr>
<td>chol</td>
<td>Cholangiocarcinoma</td>
</tr>
<tr>
<td>dlbc</td>
<td>Diffuse large B-cell</td>
</tr>
<tr>
<td>coadread</td>
<td>Colorectal</td>
</tr>
<tr>
<td>lung</td>
<td>Lung</td>
</tr>
<tr>
<td>coad</td>
<td>Colon</td>
</tr>
<tr>
<td>hg19</td>
<td>hg19</td>
</tr>
<tr>
<td>laml</td>
<td>Acute Myeloid Leukemia</td>
</tr>
<tr>
<td>read</td>
<td>Rectal</td>
</tr>
<tr>
<td>ucs</td>
<td>Uterine carcinosarcoma</td>
</tr>
<tr>
<td>uvm</td>
<td>Uveal melanoma</td>
</tr>
<tr>
<td>thym</td>
<td>Thymoma</td>
</tr>
<tr>
<td>tgct</td>
<td>Testicular germ cell</td>
</tr>
<tr>
<td>pcpg</td>
<td>Pheochromocytoma &amp; Paraganglioma</td>
</tr>
<tr>
<td>ov</td>
<td>Ovarian</td>
</tr>
<tr>
<td>meso</td>
<td>Mesothelioma</td>
</tr>
</tbody></table>

<h3 id="source">source</h3>

<p><a target="_blank" href='https://gdc.cancer.gov/'>TCGA</a></p>

<h3 id="clinical">clinical</h3>

<p>Key words to describe clinical collections for each disease type</p>

<table><thead>
<tr>
<th>clinical collection type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>events</td>
<td>clinical events collection organized by patient</td>
</tr>
<tr>
<td>patient</td>
<td>patient collection for each disease type</td>
</tr>
<tr>
<td>drug</td>
<td>chemo or other medicine administration records</td>
</tr>
<tr>
<td>newTumor</td>
<td>new tumor event records for possible patients</td>
</tr>
<tr>
<td>otherMalignancy</td>
<td>other maliganancy records for possible patients</td>
</tr>
<tr>
<td>radiation</td>
<td>radiation administration records</td>
</tr>
<tr>
<td>followUp</td>
<td>possible follow-up records</td>
</tr>
<tr>
<td>newTumor-followUp</td>
<td>possible follow-up records for the new tumor events</td>
</tr>
</tbody></table>

<h3 id="molecular">molecular</h3>

<table><thead>
<tr>
<th>molecular collection source</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>broad</td>
<td><a target="_blank" href='https://gdac.broadinstitute.org//'>Broad Firehose</a></td>
</tr>
<tr>
<td>cBio</td>
<td><a target="_blank" href='http://www.cbioportal.org/'>cBioPortal</a></td>
</tr>
<tr>
<td>ucsc-PNAS</td>
<td><a target="_blank" href='http://www.pnas.org/content/113/19/5394.full/'>Publication</a></td>
</tr>
<tr>
<td>ucsc</td>
<td><a target="_blank" href='https://genome-cancer.ucsc.edu/'>UCSC</a></td>
</tr>
<tr>
<td>bradleyLab</td>
<td><a target="_blank" href='www.fredhutch.org/en/labs/labs-import/bradley-lab.html/'>Bradley Lab</a></td>
</tr>
<tr>
<td>demo</td>
<td><a target="_blank" href='oncoscape.sttrcancer.org/'>Demo data source</a></td>
</tr>
</tbody></table>

<table><thead>
<tr>
<th>molecular collection type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>mut</td>
<td>non-synonymous mutations representated as strings in this collection</td>
</tr>
<tr>
<td>mut01</td>
<td>non-synonymous mutations representated as binary values in this collection</td>
</tr>
<tr>
<td>methylation</td>
<td>DNA methlyation data</td>
</tr>
<tr>
<td>cnv</td>
<td>DNA copy-number data represented as Gistic score</td>
</tr>
<tr>
<td>psi</td>
<td>percentage spliced in (PSI, Î¨) values in RNA splicing data</td>
</tr>
<tr>
<td>rna</td>
<td>mRNA and microRNA expression data</td>
</tr>
<tr>
<td>protein</td>
<td>protein-level and phosphoprotein level (RPPA) data</td>
</tr>
<tr>
<td>facs</td>
<td>Fluorescence-activated cell sorting data</td>
</tr>
</tbody></table>

<h3 id="category">category</h3>

<table><thead>
<tr>
<th>category collection source</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>tcga</td>
<td><a target="_blank" href='https://gdc.cancer.gov/'>TCGA</a></td>
</tr>
<tr>
<td>hgnc</td>
<td><a target="_blank" href='http://www.genenames.org/'>HUGO Gene Nomenclature Committee</a></td>
</tr>
<tr>
<td>orgHs</td>
<td><a target="_blank" href='https://bioconductor.org/packages/release/data/annotation/html/org.Hs.eg.db.html/'>Genome wide annotation for Human</a></td>
</tr>
</tbody></table>

<table><thead>
<tr>
<th>category collection type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>color</td>
<td>a collection for coloring in Oncoscape application</td>
</tr>
<tr>
<td>genesets</td>
<td>a collection of multiple genesets with specific genes in each set</td>
</tr>
</tbody></table>

<h3 id="calculated">calculated</h3>

<table><thead>
<tr>
<th>calculated collection source</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>ucsc</td>
<td><a target="_blank" href='https://genome-cancer.ucsc.edu/'>UCSC</a></td>
</tr>
<tr>
<td>broad</td>
<td><a target="_blank" href='https://gdac.broadinstitute.org//'>Broad Firehose</a></td>
</tr>
</tbody></table>

<table><thead>
<tr>
<th>calculated collection type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>pcaScores</td>
<td>calculated PCA scores for a specific data with specific genesets</td>
</tr>
<tr>
<td>pcaLoadings</td>
<td>pre-calculated PCA scores for a specific data with specific genesets</td>
</tr>
<tr>
<td>mds</td>
<td>Multidimensional Scaling data for a specific data with specific genesets</td>
</tr>
</tbody></table>

<h3 id="edges">edges</h3>

<p>edges names are collections of genesets: </p>

<table><thead>
<tr>
<th>edges geneset</th>
<th>geneset information</th>
</tr>
</thead><tbody>
<tr>
<td>TCGA GBM classifiers</td>
<td>hg19_genesets_hgnc_import[0]</td>
</tr>
<tr>
<td>Marker genes 545</td>
<td>hg19_genesets_hgnc_import[1]</td>
</tr>
<tr>
<td>TCGA pancan mutated</td>
<td>hg19_genesets_hgnc_import[2]</td>
</tr>
<tr>
<td>oncoVogel274</td>
<td>hg19_genesets_hgnc_import[3]</td>
</tr>
<tr>
<td>Oncoplex</td>
<td>hg19_genesets_hgnc_import[4]</td>
</tr>
<tr>
<td>OSCC Chen 131 probes</td>
<td>hg19_genesets_hgnc_import[5]</td>
</tr>
<tr>
<td>OSCC Chen 9 genes</td>
<td>hg19_genesets_hgnc_import[6]</td>
</tr>
</tbody></table>

<table><thead>
<tr>
<th>edges collection source</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>broad</td>
<td><a target="_blank" href='https://gdac.broadinstitute.org//'>Broad Firehose</a></td>
</tr>
<tr>
<td>ucsc</td>
<td><a target="_blank" href='https://genome-cancer.ucsc.edu/'>UCSC</a></td>
</tr>
</tbody></table>

<h2 id="from-collection-perspective">From Collection Perspective</h2>

<p>Key words to describe manifest collection</p>

<table><thead>
<tr>
<th>Key</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>dataset</td>
<td>disease code used in Oncoscape naming system</td>
</tr>
<tr>
<td>dataType</td>
<td>all possible data type for each collection</td>
</tr>
<tr>
<td>date</td>
<td>processed date</td>
</tr>
<tr>
<td>source</td>
<td>data source</td>
</tr>
<tr>
<td>process</td>
<td>process that used to derive this collection</td>
</tr>
<tr>
<td>processName</td>
<td>name of the process that used to derive this collection</td>
</tr>
<tr>
<td>parent</td>
<td>the collection(s) used to derive this collection</td>
</tr>
</tbody></table>

<h3 id="manifest-dataset">Manifest Dataset</h3>

<p>This field is identical to lookup_oncoscape_datasource&rsquo;s disease field.</p>

<table><thead>
<tr>
<th>Disease Code</th>
<th>Disease Name</th>
</tr>
</thead><tbody>
<tr>
<td>brain</td>
<td>Brain</td>
</tr>
<tr>
<td>brca</td>
<td>Breast</td>
</tr>
<tr>
<td>esca</td>
<td>Esophageal</td>
</tr>
<tr>
<td>gbm</td>
<td>Glioblastoma</td>
</tr>
<tr>
<td>hnsc</td>
<td>Head and Neck</td>
</tr>
<tr>
<td>kich</td>
<td>Kidney chromophobe</td>
</tr>
<tr>
<td>KIRC</td>
<td>Kidney renal clear cell</td>
</tr>
<tr>
<td>kirp</td>
<td>Kidney renal papillary cell</td>
</tr>
<tr>
<td>lgg</td>
<td>Lower grade glioma</td>
</tr>
<tr>
<td>lihc</td>
<td>Liver</td>
</tr>
<tr>
<td>luad</td>
<td>Lung adenocarcinoma</td>
</tr>
<tr>
<td>lusc</td>
<td>Lung squamous cell</td>
</tr>
<tr>
<td>sarc</td>
<td>Sarcoma</td>
</tr>
<tr>
<td>paad</td>
<td>Pancreas</td>
</tr>
<tr>
<td>prad</td>
<td>Prostate</td>
</tr>
<tr>
<td>skcm</td>
<td>Skin cutaneous melanoma</td>
</tr>
<tr>
<td>stad</td>
<td>Stomach</td>
</tr>
<tr>
<td>thca</td>
<td>Thyroid carcinoma</td>
</tr>
<tr>
<td>ucec</td>
<td>Uterine corpus endometrial</td>
</tr>
<tr>
<td>acc</td>
<td>Adrenocortical carcinoma</td>
</tr>
<tr>
<td>blca</td>
<td>Bladder urothelial carcinoma</td>
</tr>
<tr>
<td>cesc</td>
<td>Cervical</td>
</tr>
<tr>
<td>chol</td>
<td>Cholangiocarcinoma</td>
</tr>
<tr>
<td>dlbc</td>
<td>Diffuse large B-cell</td>
</tr>
<tr>
<td>coadread</td>
<td>Colorectal</td>
</tr>
<tr>
<td>lung</td>
<td>Lung</td>
</tr>
<tr>
<td>coad</td>
<td>Colon</td>
</tr>
<tr>
<td>hg19</td>
<td>hg19</td>
</tr>
<tr>
<td>laml</td>
<td>Acute Myeloid Leukemia</td>
</tr>
<tr>
<td>read</td>
<td>Rectal</td>
</tr>
<tr>
<td>ucs</td>
<td>Uterine carcinosarcoma</td>
</tr>
<tr>
<td>uvm</td>
<td>Uveal melanoma</td>
</tr>
<tr>
<td>thym</td>
<td>Thymoma</td>
</tr>
<tr>
<td>tgct</td>
<td>Testicular germ cell</td>
</tr>
<tr>
<td>pcpg</td>
<td>Pheochromocytoma &amp; Paraganglioma</td>
</tr>
<tr>
<td>ov</td>
<td>Ovarian</td>
</tr>
<tr>
<td>meso</td>
<td>Mesothelioma</td>
</tr>
<tr>
<td>kirc</td>
<td>Kidney renal clear cell</td>
</tr>
</tbody></table>

<h3 id="all-the-datatypes-within-database">All the datatypes within Database</h3>

<table><thead>
<tr>
<th>collection type</th>
<th>Annotation</th>
</tr>
</thead><tbody>
<tr>
<td>color</td>
<td>a collection for coloring in Oncoscape application</td>
</tr>
<tr>
<td>mut</td>
<td>non-synonymous mutations representated as strings in this collection</td>
</tr>
<tr>
<td>mut01</td>
<td>non-synonymous mutations representated as binary values in this collection</td>
</tr>
<tr>
<td>events</td>
<td>clinical events collection organized by patient</td>
</tr>
<tr>
<td>patient</td>
<td>patient collection for each disease type</td>
</tr>
<tr>
<td>drug</td>
<td>chemo or other medicine administration records</td>
</tr>
<tr>
<td>newTumor</td>
<td>new tumor event records for possible patients</td>
</tr>
<tr>
<td>otherMalignancy</td>
<td>other maliganancy records for possible patients</td>
</tr>
<tr>
<td>radiation</td>
<td>radiation administration records</td>
</tr>
<tr>
<td>followUp</td>
<td>possible follow-up records</td>
</tr>
<tr>
<td>newTumor-followUp</td>
<td>possible follow-up records for the new tumor events</td>
</tr>
<tr>
<td>genesets</td>
<td>a collection of multiple genesets with specific genes in each set</td>
</tr>
<tr>
<td>methylation</td>
<td>DNA methlyation data</td>
</tr>
<tr>
<td>rna</td>
<td>mRNA and microRNA expression data</td>
</tr>
<tr>
<td>protein</td>
<td>protein-level and phosphoprotein level (RPPA) data</td>
</tr>
<tr>
<td>psi</td>
<td>percentage spliced in (PSI, Î¨) values in RNA splicing data</td>
</tr>
<tr>
<td>facs</td>
<td>Fluorescence-activated cell sorting data</td>
</tr>
<tr>
<td>cnv</td>
<td>DNA copy-number data represented as Gistic score</td>
</tr>
<tr>
<td>annotation</td>
<td>annotation files for a specific data type, for instance RNA splicing</td>
</tr>
<tr>
<td>chromosome</td>
<td>annoataion for chromosomes</td>
</tr>
<tr>
<td>genes</td>
<td>annotation data used to annotate genes</td>
</tr>
<tr>
<td>centromere</td>
<td>centromere position for each chromosome on human genome</td>
</tr>
<tr>
<td>pcaScores</td>
<td>calculated PCA scores for a specific data with specific genesets</td>
</tr>
<tr>
<td>pcaLoadings</td>
<td>pre-calculated PCA scores for a specific data with specific genesets</td>
</tr>
<tr>
<td>mds</td>
<td>Multidimensional Scaling data for a specific data with specific genesets</td>
</tr>
<tr>
<td>edges</td>
<td>derived collection to describe edges between genes and patients use for Markers and Patients (one of the Oncoscape tools)</td>
</tr>
<tr>
<td>ptDegree</td>
<td>derived collection to describe the weight of patients based on on the number of data points use for Markers and Patients (one of the Oncoscape tools)</td>
</tr>
<tr>
<td>geneDegree</td>
<td>derived collection to describe the weight of genes based on on the number of data points use for Markers and Patients (one of the Oncoscape tools)</td>
</tr>
</tbody></table>

<h2 id="genesets-details">Genesets Details:</h2>

<table><thead>
<tr>
<th>Key</th>
<th>description</th>
<th>number of genes</th>
</tr>
</thead><tbody>
<tr>
<td>TCGA_GBM_classifiers</td>
<td><a target="_blank" href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2818769/'>TCGA_GBM_classifiers</a></td>
<td>840</td>
</tr>
<tr>
<td>Marker_genes_545</td>
<td>The 545-gene set is all the genes that are recurrently impacted in TCGA gliomas.</td>
<td>545</td>
</tr>
<tr>
<td>TCGA_pancan_mutated</td>
<td>all the genes recurrently mutated across all TCGA cancers (note that it may not be up to date for all the TCGA samples currently available</td>
<td>64</td>
</tr>
<tr>
<td>oncoVogel274</td>
<td>The OncoVogel274 gene set comes from combining (an old version of Oncoplex) with genes from this (famous) paper by <a target="_blank" href='https://www.ncbi.nlm.nih.gov/pubmed/23539594/'>Vogelstein</a></td>
<td>274</td>
</tr>
<tr>
<td>Oncoplex</td>
<td><a target="_blank" href='http://tests.labmed.washington.edu/UW-OncoPlex/'>UW Oncoplex Cancer Panel</a></td>
<td>263</td>
</tr>
<tr>
<td>OSCC_Chen_131_probes</td>
<td><a target="_blank" href='https://www.ncbi.nlm.nih.gov/pubmed/18669583/'>OSCC_Chen_131_probes</a></td>
<td>109</td>
</tr>
<tr>
<td>OSCC_Chen_9_genes</td>
<td><a target="_blank" href='https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4527746//'>OSCC_Chen_9_genes</a></td>
<td>9</td>
</tr>
</tbody></table>

          <h1 id="rest-api-queries">Rest API Queries</h1>

<h2 id="example-to-access-one-collection-from-browser">Example to access one collection from browser</h2>

<h3 id="http-request">HTTP Request</h3>

<p>Collections are accessable at the host: http://dev.oncoscape.sttrcancer.io/api/</p>

<p>The endpoint of oncoscape API is a unique URL. Every endpoint points to a unique collection.
Below lists more details of the organization of the Oncoscape Mongo Database and the collections organized by disease type.</p>

<p><code class="prettyprint">GET http://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical/</code></p>

<blockquote>
<p>Here we only show the first record in gbm_patient_tcga_clinical</p>
</blockquote>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"57c9a095d482221d835e6018"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"patient_ID"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA-02-0001-01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"history_lgg_dx_of_brain_tissue"</span><span class="p">:</span><span class="w"> </span><span class="kc">false</span><span class="p">,</span><span class="w">
    </span><span class="nt">"prospective_collection"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"retrospective_collection"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"gender"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FEMALE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_birth"</span><span class="p">:</span><span class="w"> </span><span class="mi">-16179</span><span class="p">,</span><span class="w">
    </span><span class="nt">"race"</span><span class="p">:</span><span class="w"> </span><span class="s2">"WHITE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"ethnicity"</span><span class="p">:</span><span class="w"> </span><span class="s2">"NOT HISPANIC OR LATINO"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"history_other_malignancy"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"history_neoadjuvant_treatment"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
    </span><span class="nt">"diagnosis_year"</span><span class="p">:</span><span class="w"> </span><span class="mi">1009872000</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pathologic_method"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TUMOR RESECTION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pathologic_method.1"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status_vital"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DEAD"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_last_contact"</span><span class="p">:</span><span class="w"> </span><span class="mi">279</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_death"</span><span class="p">:</span><span class="w"> </span><span class="mi">358</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status_tumor"</span><span class="p">:</span><span class="w"> </span><span class="s2">"WITH TUMOR"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"KPS"</span><span class="p">:</span><span class="w"> </span><span class="mi">80</span><span class="p">,</span><span class="w">
    </span><span class="nt">"ECOG"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"encounter_type"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"radiation_treatment_adjuvant"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pharmaceutical_tx_adjuvant"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"treatment_outcome_first_course"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"new_tumor_event_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"age_at_initial_pathologic_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="mi">44</span><span class="p">,</span><span class="w">
    </span><span class="nt">"anatomic_organ_subdivision"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
    </span><span class="nt">"disease_code"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"histologic_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"icd_10"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C71.9"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"icd_3_histology"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9440/3"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"icd_3"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C71.9"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"tissue_source_site_code"</span><span class="p">:</span><span class="w"> </span><span class="s2">"02"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"tumor_tissue_site"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BRAIN"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<h2 id="query-collection-from-browser">Query Collection from Browser</h2>

<h3 id="http-request">HTTP Request</h3>

<blockquote>
<p>Here we show the first two records that meet the below criteria: gender is male, race is white. We have skipped the first five records from the results. And we only show three fields (patient_ID, gender and race.</p>
</blockquote>
<pre class="highlight plaintext"><code>
[
    {
        "_id": "57c9a096d482221d835e6025",
        "patient_ID": "TCGA-02-0024-01",
        "gender": "MALE",
        "race": "WHITE",
        "histologic_diagnosis": "GLIOBLASTOMA MULTIFORME TREATED PRIMARY"
    },
    {
        "_id": "57c9a096d482221d835e6026",
        "patient_ID": "TCGA-02-0025-01",
        "gender": "MALE",
        "race": "WHITE",
        "histologic_diagnosis": "GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"
    }
]
</code></pre>

<p>Filter by gender and race and only show the selected fields</p>

<p><code class="prettyprint">GET http://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical/?q={&quot;gender&quot;:&quot;MALE&quot;, &quot;race&quot;:&quot;WHITE&quot;,&quot;$fields&quot;:[&quot;gender&quot;,&quot;race&quot;,&quot;patient_ID&quot;],&quot;$skip&quot;:5,&quot;$limit&quot;:2}</code></p>

<p>only show gender, race and patient_ID</p>

<p><code class="prettyprint">&quot;$fields&quot;:[&quot;gender&quot;,&quot;race&quot;,&quot;patient_ID&quot;]</code></p>

<p>skip the first five records</p>

<p><code class="prettyprint">&quot;$skip&quot;:5</code></p>

<p>limit the final output to two records.</p>

<p><code class="prettyprint">&quot;$limit&quot;:2</code></p>

<h2 id="fetch-json-formatted-data-using-programming-languages">Fetch JSON-Formatted Data Using Programming Languages</h2>

<blockquote>
<p>Fetch JSON formatted data using R, Python, or javascript</p>
</blockquote>
<pre class="highlight javascript"><code><span class="kd">var</span> <span class="nx">collection</span> <span class="o">=</span> <span class="s2">"gbm_patient_tcga_clinical"</span><span class="p">;</span>
<span class="kd">var</span> <span class="nx">url</span> <span class="o">=</span> <span class="s2">"https://dev.oncoscape.sttrcancer.io/api/"</span> <span class="o">+</span> <span class="nx">collection</span> <span class="o">+</span> <span class="s2">"/?q="</span><span class="p">;</span>
<span class="nx">$</span><span class="p">.</span><span class="nx">get</span><span class="p">(</span><span class="nx">url</span><span class="p">,</span> <span class="kd">function</span><span class="p">(</span><span class="nx">data</span><span class="p">)</span> <span class="p">{</span>
     <span class="kd">var</span> <span class="nx">field_names</span> <span class="o">=</span> <span class="nb">Object</span><span class="p">.</span><span class="nx">keys</span><span class="p">(</span><span class="nx">data</span><span class="p">[</span><span class="mi">0</span><span class="p">]);</span>
     <span class="kd">var</span> <span class="nx">count</span> <span class="o">=</span> <span class="nx">data</span><span class="p">.</span><span class="nx">length</span><span class="p">;</span>
     <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="s2">"fields name of the first records: "</span> <span class="o">+</span> <span class="nx">field_names</span><span class="p">);</span>
     <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="s2">"counts: "</span> <span class="o">+</span> <span class="nx">count</span><span class="p">);</span>
  <span class="p">});</span>
</code></pre>
<pre class="highlight r"><code><span class="n">install.packages</span><span class="p">(</span><span class="s2">"jsonlite"</span><span class="p">)</span>
<span class="n">install.packages</span><span class="p">(</span><span class="s2">"curl"</span><span class="p">)</span>
<span class="n">library</span><span class="p">(</span><span class="n">jsonlite</span><span class="p">)</span>
<span class="n">library</span><span class="p">(</span><span class="n">curl</span><span class="p">)</span>
<span class="n">gbm_patient</span> <span class="o">&lt;-</span> <span class="n">fromJSON</span><span class="p">(</span><span class="s2">"https://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical"</span><span class="p">)</span>
<span class="n">str</span><span class="p">(</span><span class="n">gbm_patient</span><span class="p">,</span> <span class="n">max.level</span><span class="o">=</span><span class="m">2</span><span class="p">)</span>
<span class="s1">'data.frame'</span><span class="o">:</span> <span class="m">596</span> <span class="n">obs.</span> <span class="n">of</span>  <span class="m">33</span> <span class="n">variables</span><span class="o">:</span>
<span class="o">$</span> <span class="n">patient_ID</span>                         <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"TCGA-02-0001-01"</span> <span class="s2">"TCGA-02-0003-01"</span> <span class="s2">"TCGA-02-0004-01"</span> <span class="s2">"TCGA-02-0006-01"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">history_lgg_dx_of_brain_tissue</span>     <span class="o">:</span> <span class="n">logi</span>  <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">prospective_collection</span>             <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">retrospective_collection</span>           <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">gender</span>                             <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"FEMALE"</span> <span class="s2">"MALE"</span> <span class="s2">"MALE"</span> <span class="s2">"FEMALE"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_birth</span>                      <span class="o">:</span> <span class="n">int</span>  <span class="m">-16179</span> <span class="m">-18341</span> <span class="m">-21617</span> <span class="m">-20516</span> <span class="m">-14806</span> <span class="m">-22457</span> <span class="m">-7452</span> <span class="m">-6926</span> <span class="m">-9369</span> <span class="m">-18404</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">race</span>                               <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"WHITE"</span> <span class="s2">"WHITE"</span> <span class="s2">"WHITE"</span> <span class="s2">"WHITE"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">ethnicity</span>                          <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">history_other_malignancy</span>           <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">history_neoadjuvant_treatment</span>      <span class="o">:</span> <span class="n">logi</span>  <span class="n">TRUE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">TRUE</span> <span class="n">FALSE</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">diagnosis_year</span>                     <span class="o">:</span> <span class="n">int</span>  <span class="m">1009872000</span> <span class="m">1041408000</span> <span class="m">1009872000</span> <span class="m">1009872000</span> <span class="m">1009872000</span> <span class="m">1041408000</span> <span class="m">1009872000</span> <span class="m">1072944000</span> <span class="m">852105600</span> <span class="m">1009872000</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">pathologic_method</span>                  <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">pathologic_method</span>                  <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">status_vital</span>                       <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"DEAD"</span> <span class="s2">"DEAD"</span> <span class="s2">"DEAD"</span> <span class="s2">"DEAD"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_last_contact</span>               <span class="o">:</span> <span class="n">int</span>  <span class="m">279</span> <span class="m">144</span> <span class="m">345</span> <span class="m">558</span> <span class="m">705</span> <span class="m">322</span> <span class="m">1077</span> <span class="m">630</span> <span class="m">2512</span> <span class="m">627</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_death</span>                      <span class="o">:</span> <span class="n">int</span>  <span class="m">358</span> <span class="m">144</span> <span class="m">345</span> <span class="m">558</span> <span class="m">705</span> <span class="m">322</span> <span class="m">1077</span> <span class="m">630</span> <span class="m">2512</span> <span class="m">627</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">status_tumor</span>                       <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"WITH TUMOR"</span> <span class="s2">"WITH TUMOR"</span> <span class="s2">"WITH TUMOR"</span> <span class="s2">"WITH TUMOR"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">KPS</span>                                <span class="o">:</span> <span class="n">int</span>  <span class="m">80</span> <span class="m">100</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">100</span> <span class="m">80</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">ECOG</span>                               <span class="o">:</span> <span class="n">int</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">encounter_type</span>                     <span class="o">:</span> <span class="n">chr</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">radiation_treatment_adjuvant</span>       <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">pharmaceutical_tx_adjuvant</span>         <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">treatment_outcome_first_course</span>     <span class="o">:</span> <span class="n">chr</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">new_tumor_event_diagnosis</span>          <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">age_at_initial_pathologic_diagnosis</span><span class="o">:</span> <span class="n">int</span>  <span class="m">44</span> <span class="m">50</span> <span class="m">59</span> <span class="m">56</span> <span class="m">40</span> <span class="m">61</span> <span class="m">20</span> <span class="m">18</span> <span class="m">25</span> <span class="m">50</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">anatomic_organ_subdivision</span>         <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_diagnosis</span>                  <span class="o">:</span> <span class="n">int</span>  <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">disease_code</span>                       <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">histologic_diagnosis</span>               <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">icd_10</span>                             <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">icd_3_histology</span>                    <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"9440/3"</span> <span class="s2">"9440/3"</span> <span class="s2">"9440/3"</span> <span class="s2">"9440/3"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">icd_3</span>                              <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">tissue_source_site_code</span>            <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"02"</span> <span class="s2">"02"</span> <span class="s2">"02"</span> <span class="s2">"02"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">tumor_tissue_site</span>                  <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"BRAIN"</span> <span class="s2">"BRAIN"</span> <span class="s2">"BRAIN"</span> <span class="s2">"BRAIN"</span> <span class="k">...</span>
</code></pre>
<pre class="highlight python"><code>
<span class="o">&gt;</span> <span class="n">shell</span> <span class="n">commands</span><span class="p">:</span> <span class="n">sudo</span> <span class="n">pip</span> <span class="n">install</span> <span class="n">pymongo</span><span class="p">,</span> <span class="n">simplejson</span><span class="p">,</span> <span class="n">urllib2</span><span class="p">,</span> <span class="n">json</span>

<span class="kn">import</span> <span class="nn">urllib2</span>
<span class="kn">import</span> <span class="nn">json</span>
<span class="kn">import</span> <span class="nn">simplejson</span>
<span class="n">url</span> <span class="o">=</span> <span class="s">"https://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical"</span>
<span class="n">response</span> <span class="o">=</span> <span class="n">urlli2</span><span class="o">.</span><span class="n">urlopen</span><span class="p">(</span><span class="n">url</span><span class="p">)</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">simplejson</span><span class="o">.</span><span class="n">load</span><span class="p">(</span><span class="n">response</span><span class="p">)</span>
<span class="k">print</span> <span class="n">json</span><span class="o">.</span><span class="n">dumps</span><span class="p">(</span><span class="n">data</span><span class="p">[</span><span class="mi">0</span><span class="p">:</span><span class="mi">2</span><span class="p">],</span> <span class="n">indent</span><span class="o">=</span><span class="mi">4</span><span class="p">,</span> <span class="n">sort_keys</span><span class="o">=</span><span class="bp">True</span><span class="p">)</span>

<span class="p">[</span>
    <span class="p">{</span>
        <span class="s">"ECOG"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"KPS"</span><span class="p">:</span> <span class="mi">80</span><span class="p">,</span>
        <span class="s">"age_at_initial_pathologic_diagnosis"</span><span class="p">:</span> <span class="mi">44</span><span class="p">,</span>
        <span class="s">"anatomic_organ_subdivision"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"days_to_birth"</span><span class="p">:</span> <span class="o">-</span><span class="mi">16179</span><span class="p">,</span>
        <span class="s">"days_to_death"</span><span class="p">:</span> <span class="mi">358</span><span class="p">,</span>
        <span class="s">"days_to_diagnosis"</span><span class="p">:</span> <span class="mi">0</span><span class="p">,</span>
        <span class="s">"days_to_last_contact"</span><span class="p">:</span> <span class="mi">279</span><span class="p">,</span>
        <span class="s">"diagnosis_year"</span><span class="p">:</span> <span class="mi">1009872000</span><span class="p">,</span>
        <span class="s">"disease_code"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"encounter_type"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"ethnicity"</span><span class="p">:</span> <span class="s">"NOT HISPANIC OR LATINO"</span><span class="p">,</span>
        <span class="s">"gender"</span><span class="p">:</span> <span class="s">"FEMALE"</span><span class="p">,</span>
        <span class="s">"histologic_diagnosis"</span><span class="p">:</span> <span class="s">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span><span class="p">,</span>
        <span class="s">"history_lgg_dx_of_brain_tissue"</span><span class="p">:</span> <span class="n">false</span><span class="p">,</span>
        <span class="s">"history_neoadjuvant_treatment"</span><span class="p">:</span> <span class="n">true</span><span class="p">,</span>
        <span class="s">"history_other_malignancy"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"icd_10"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3_histology"</span><span class="p">:</span> <span class="s">"9440/3"</span><span class="p">,</span>
        <span class="s">"new_tumor_event_diagnosis"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"pathologic_method"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"patient_ID"</span><span class="p">:</span> <span class="s">"TCGA-02-0001-01"</span><span class="p">,</span>
        <span class="s">"pharmaceutical_tx_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"prospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"race"</span><span class="p">:</span> <span class="s">"WHITE"</span><span class="p">,</span>
        <span class="s">"radiation_treatment_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"retrospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"status_tumor"</span><span class="p">:</span> <span class="s">"WITH TUMOR"</span><span class="p">,</span>
        <span class="s">"status_vital"</span><span class="p">:</span> <span class="s">"DEAD"</span><span class="p">,</span>
        <span class="s">"tissue_source_site_code"</span><span class="p">:</span> <span class="s">"02"</span><span class="p">,</span>
        <span class="s">"treatment_outcome_first_course"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"tumor_tissue_site"</span><span class="p">:</span> <span class="s">"BRAIN"</span>
    <span class="p">},</span>
    <span class="p">{</span>
        <span class="s">"ECOG"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"KPS"</span><span class="p">:</span> <span class="mi">100</span><span class="p">,</span>
        <span class="s">"age_at_initial_pathologic_diagnosis"</span><span class="p">:</span> <span class="mi">50</span><span class="p">,</span>
        <span class="s">"anatomic_organ_subdivision"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"days_to_birth"</span><span class="p">:</span> <span class="o">-</span><span class="mi">18341</span><span class="p">,</span>
        <span class="s">"days_to_death"</span><span class="p">:</span> <span class="mi">144</span><span class="p">,</span>
        <span class="s">"days_to_diagnosis"</span><span class="p">:</span> <span class="mi">0</span><span class="p">,</span>
        <span class="s">"days_to_last_contact"</span><span class="p">:</span> <span class="mi">144</span><span class="p">,</span>
        <span class="s">"diagnosis_year"</span><span class="p">:</span> <span class="mi">1041408000</span><span class="p">,</span>
        <span class="s">"disease_code"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"encounter_type"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"ethnicity"</span><span class="p">:</span> <span class="s">"NOT HISPANIC OR LATINO"</span><span class="p">,</span>
        <span class="s">"gender"</span><span class="p">:</span> <span class="s">"MALE"</span><span class="p">,</span>
        <span class="s">"histologic_diagnosis"</span><span class="p">:</span> <span class="s">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span><span class="p">,</span>
        <span class="s">"history_lgg_dx_of_brain_tissue"</span><span class="p">:</span> <span class="n">false</span><span class="p">,</span>
        <span class="s">"history_neoadjuvant_treatment"</span><span class="p">:</span> <span class="n">false</span><span class="p">,</span>
        <span class="s">"history_other_malignancy"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"icd_10"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3_histology"</span><span class="p">:</span> <span class="s">"9440/3"</span><span class="p">,</span>
        <span class="s">"new_tumor_event_diagnosis"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"pathologic_method"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"patient_ID"</span><span class="p">:</span> <span class="s">"TCGA-02-0003-01"</span><span class="p">,</span>
        <span class="s">"pharmaceutical_tx_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"prospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"race"</span><span class="p">:</span> <span class="s">"WHITE"</span><span class="p">,</span>
        <span class="s">"radiation_treatment_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"retrospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"status_tumor"</span><span class="p">:</span> <span class="s">"WITH TUMOR"</span><span class="p">,</span>
        <span class="s">"status_vital"</span><span class="p">:</span> <span class="s">"DEAD"</span><span class="p">,</span>
        <span class="s">"tissue_source_site_code"</span><span class="p">:</span> <span class="s">"02"</span><span class="p">,</span>
        <span class="s">"treatment_outcome_first_course"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"tumor_tissue_site"</span><span class="p">:</span> <span class="s">"BRAIN"</span>
    <span class="p">}</span>
<span class="p">]</span>
</code></pre>

<p>Users can access json-formatted data using URL link.</p>

<p>Here we show the example to access one collection using four different languages.</p>

<h1 id="collections-by-disease">Collections by Disease</h1>

<h2 id="brain-lower-grade-glioma-amp-glioblastoma-multiforme">BRAIN - Lower Grade Glioma &amp; Glioblastoma multiforme</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>brain_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_events_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_patient_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_drug_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_radiation_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>brain_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>brain_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>brain_cnv_ucsc-pnas_gistic</td>
<td>molecular</td>
<td>ucsc-PNAS</td>
<td>cnv</td>
</tr>
<tr>
<td>brain_mut01_ucsc-pnas_import</td>
<td>molecular</td>
<td>ucsc-PNAS</td>
<td>mut01</td>
</tr>
<tr>
<td>brain_cnv_ucsc_gistic</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>brain_mut01_ucsc_import</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>brain_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>brain_mut_cbio_wxs</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>brain_mut01_cbio_wxs</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brain_cnv_ucsc_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1090</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brain_mut01_ucsc_import"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">818</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>brain_cnv_ucsc_gistic</td>
<td>ucsc</td>
<td>cnv</td>
<td>1090</td>
<td></td>
</tr>
<tr>
<td>brain_mut01_ucsc_import</td>
<td>ucsc</td>
<td>mut01</td>
<td>818</td>
<td></td>
</tr>
</tbody></table>

<h2 id="brca-breast-invasive-carcinoma">BRCA - Breast invasive carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>brca_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_events_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_patient_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_drug_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_radiation_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_followup_tcga_v4p0</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>brca_psi_bradleyLab_miso</td>
<td>molecular</td>
<td>bradleyLab</td>
<td>psi</td>
</tr>
<tr>
<td>brca_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>brca_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>brca_mut_cbio_wxs</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>brca_mut01_cbio_wxs</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>brca_rna_cbio_agilent-median-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>brca_rna_cbio_seq-median-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>brca_protein_cbio_rppa-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>brca_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>brca_mut01_ucsc_mutationcuratedwustlgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1079</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 1079 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_mut_cbio_wxs"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">980</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_methylation_cbio_hm27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">315</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27) beta-values for genes in 342 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_methylation_cbio_hm450"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">737</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450) beta-values for genes in 833 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_protein_cbio_rppa-zscore"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">409</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/BRCA/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1079</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA breast invasive carcinoma (BRCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>brca_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>1079</td>
<td>Putative copy-number calls on 1079 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>brca_mut_cbio_wxs</td>
<td>cbio</td>
<td>mut</td>
<td>980</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm27</td>
<td>cbio</td>
<td>methylation</td>
<td>315</td>
<td>Methylation (HM27) beta-values for genes in 342 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm450</td>
<td>cbio</td>
<td>methylation</td>
<td>737</td>
<td>Methylation (HM450) beta-values for genes in 833 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>brca_protein_cbio_rppa-zscore</td>
<td>cbio</td>
<td>protein</td>
<td>409</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
<tr>
<td>brca_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>1079</td>
<td>TCGA breast invasive carcinoma (BRCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="esca-esophageal-carcinoma">ESCA - Esophageal carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>esca_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>esca_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>esca_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>esca_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>esca_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>esca_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"esca_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/ESCA/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">184</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA esophageal carcinoma (ESCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>esca_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>184</td>
<td>TCGA esophageal carcinoma (ESCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="gbm-glioblastoma-multiforme">GBM - Glioblastoma multiforme</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>gbm_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_newtumor-followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>gbm_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>gbm_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>gbm_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>gbm_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gbm_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/GBM/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">577</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA glioblastoma multiforme (GBM) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>gbm_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>577</td>
<td>TCGA glioblastoma multiforme (GBM) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="hnsc-head-and-neck-squamous-cell-carcinoma">HNSC - Head and Neck squamous cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>hnsc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_followup_tcga_v4p8</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_newtumor-followup_tcga_v4p8</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>hnsc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>hnsc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>hnsc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>hnsc_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>hnsc_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>hnsc_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>hnsc_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>hnsc_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>hnsc_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>hnsc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/HNSC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">522</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA head &amp; neck squamous cell carcinoma (HNSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/hnsc/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">509</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA head &amp; neck squamous cell carcinoma (HNSC) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">522</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 522 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">306</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">306</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">212</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>hnsc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>522</td>
<td>TCGA head &amp; neck squamous cell carcinoma (HNSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>hnsc_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>509</td>
<td>TCGA head &amp; neck squamous cell carcinoma (HNSC) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>hnsc_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>522</td>
<td>Putative copy-number calls on 522 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>hnsc_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>306</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>hnsc_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>306</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>hnsc_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>212</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="kich-kidney-chromophobe">KICH - Kidney Chromophobe</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>kich_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_followup_tcga_v4p4</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_newtumor-followup_tcga_v4p4</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kich_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>kich_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>kich_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>kich_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="kirc-kidney-renal-clear-cell-carcinoma">KIRC - Kidney renal clear cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>kirc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirc_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>kirc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
</tbody></table>

<h2 id="kirp-kidney-renal-papillary-cell-carcinoma">KIRP - Kidney renal papillary cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>kirp_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirp_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirp_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirp_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirp_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirp_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>kirp_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>kirp_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>kirp_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>kirp_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="lgg-brain-lower-grade-glioma">LGG - Brain Lower Grade Glioma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lgg_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>lgg_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>lgg_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>lgg_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>lgg_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lgg_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LGG/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">513</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA brain lower grade glioma (LGG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lgg_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>lgg_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>513</td>
<td>TCGA brain lower grade glioma (LGG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>lgg_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="lihc-undefined">LIHC - undefined</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lihc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lihc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>lihc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>lihc_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>lihc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="luad-lung-adenocarcinoma">LUAD - Lung adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>luad_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>luad_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>luad_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>luad_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>luad_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>luad_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>luad_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>luad_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>luad_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>luad_rna_cbio_agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>luad_rna_cbio_seq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>luad_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>luad_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>luad_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LUAD/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">515</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung adenocarcinoma (LUAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/luad/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">543</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung adenocarcinoma (LUAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">515</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 515 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">229</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">229</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">181</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>luad_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>515</td>
<td>TCGA lung adenocarcinoma (LUAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>luad_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>543</td>
<td>TCGA lung adenocarcinoma (LUAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>luad_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>515</td>
<td>Putative copy-number calls on 515 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>luad_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>229</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>luad_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>229</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>luad_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>181</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="lusc-lung-squamous-cell-carcinoma">LUSC - Lung squamous cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lusc_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lusc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>lusc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>lusc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>lusc_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>lusc_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>lusc_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>lusc_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>lusc_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lusc_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lusc_rna_cbio_agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_rna_cbio_seq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_rna_cbio_u133</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>lusc_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LUSC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">501</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung squamous cell carcinoma (LUSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">32</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 501 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">178</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">178</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>lusc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>501</td>
<td>TCGA lung squamous cell carcinoma (LUSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>lusc_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>lusc_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>32</td>
<td>Putative copy-number calls on 501 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>lusc_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>178</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>lusc_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>178</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>lusc_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>0</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="sarc-sarcoma">SARC - Sarcoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>sarc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>sarc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>sarc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>sarc_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>sarc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"sarc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/SARC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">257</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"   cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA sarcoma (SARC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>sarc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>257</td>
<td>TCGA sarcoma (SARC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="paad-pancreatic-adenocarcinoma">PAAD - Pancreatic adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>paad_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_followup_tcga_v4p4</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_newtumor-followup_tcga_v4p4</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>paad_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>paad_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>paad_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>paad_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>paad_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>paad_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>paad_rna_cbio_seq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>paad_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>paad_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/PAAD/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">184</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA pancreatic adenocarcinoma (PAAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/paad/gsc/hgsc.bcm.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">131</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA pancreatic adenocarcinoma (PAAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">195</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 184 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">91</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">91</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>paad_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>184</td>
<td>TCGA pancreatic adenocarcinoma (PAAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>paad_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>131</td>
<td>TCGA pancreatic adenocarcinoma (PAAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method.</td>
</tr>
<tr>
<td>paad_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>195</td>
<td>Putative copy-number calls on 184 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>paad_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>91</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>paad_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>91</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="prad-prostate-adenocarcinoma">PRAD - Prostate adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>prad_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>prad_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>prad_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>prad_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>prad_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>prad_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>prad_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>prad_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>prad_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>prad_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>prad_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>prad_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>prad_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>prad_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>prad_rna_cbio_agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>prad_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>prad_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>prad_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/PRAD/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">492</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"   cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA prostate adenocarcinoma (PRAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/prad/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">425</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA prostate adenocarcinoma (PRAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">492</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 492 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">261</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">261</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">164</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>prad_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>492</td>
<td>TCGA prostate adenocarcinoma (PRAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>prad_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>425</td>
<td>TCGA prostate adenocarcinoma (PRAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>prad_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>492</td>
<td>Putative copy-number calls on 492 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>prad_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>261</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>prad_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>261</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>prad_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>164</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="skcm-skin-cutaneous-melanoma">SKCM - Skin Cutaneous Melanoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>skcm_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>skcm_followup_tcga_v2p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>skcm_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>skcm_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>skcm_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>skcm_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>skcm_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>skcm_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>skcm_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>skcm_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="stad-stomach-adenocarcinoma">STAD - Stomach adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>stad_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>stad_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>stad_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>stad_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>stad_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>stad_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>stad_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>stad_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>stad_mut01_ucsc_mutationbcmgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>stad_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>stad_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"stad_mut01_ucsc_mutationbcmgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/stad/gsc/hgsc.bcm.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">379</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA stomach adenocarcinoma (STAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>stad_mut01_ucsc_mutationbcmgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>379</td>
<td>TCGA stomach adenocarcinoma (STAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="thca-thyroid-carcinoma">THCA - Thyroid carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>thca_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thca_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>thca_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>thca_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>thca_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="ucec-uterine-corpus-endometrial-carcinoma">UCEC - Uterine Corpus Endometrial Carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>ucec_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_followup_tcga_v2p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucec_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>ucec_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>ucec_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>ucec_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="acc-adrenocortical-carcinoma">ACC - Adrenocortical carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>acc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>acc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>acc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>acc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>acc_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>acc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"acc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/ACC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">90</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA adrenocortical carcinoma (ACC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>acc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>90</td>
<td>TCGA adrenocortical carcinoma (ACC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="blca-bladder-urothelial-carcinoma">BLCA - Bladder Urothelial Carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>blca_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>blca_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>blca_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>blca_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>blca_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>blca_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"blca_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/BLCA/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA bladder urothelial carcinoma (BLCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"blca_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/blca/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA bladder urothelial carcinoma (BLCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>blca_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>408</td>
<td>TCGA bladder urothelial carcinoma (BLCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>blca_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>238</td>
<td>TCGA bladder urothelial carcinoma (BLCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="cesc-cervical-squamous-cell-carcinoma-and-endocervical-adenocarcinoma">CESC - Cervical squamous cell carcinoma and endocervical adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>cesc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>cesc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>cesc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>cesc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>cesc_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>cesc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cesc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/CESC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">295</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>cesc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>295</td>
<td>TCGA cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="chol-cholangiocarcinoma">CHOL - Cholangiocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>chol_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>chol_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>chol_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>chol_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>chol_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>chol_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"chol_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/CHOL/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">36</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA cholangiocarcinoma (CHOL) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>chol_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>36</td>
<td>TCGA cholangiocarcinoma (CHOL) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="dlbc-lymphoid-neoplasm-diffuse-large-b-cell-lymphoma">DLBC - Lymphoid Neoplasm Diffuse Large B-cell Lymphoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>dlbc_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>dlbc_followup_tcga_v4p4</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>dlbc_newtumor-followup_tcga_v4p4</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>dlbc_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>dlbc_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>dlbc_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>dlbc_mut_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut</td>
</tr>
<tr>
<td>dlbc_mut01_broad_mutsig2</td>
<td>molecular</td>
<td>broad</td>
<td>mut01</td>
</tr>
<tr>
<td>dlbc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>dlbc_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>dlbc_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"dlbc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/DLBC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">48</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lymphoid neoplasm diffuse large B-cell lymphoma (DLBC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>dlbc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>48</td>
<td>TCGA lymphoid neoplasm diffuse large B-cell lymphoma (DLBC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="coadread-colon-adenocarcinoma-amp-rectum-adenocarcinoma">COADREAD - Colon adenocarcinoma &amp; Rectum adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>coadread_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coadread_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>coadread_mut01_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>coadread_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>coadread_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>coadread_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>coadread_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>coadread_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>coadread_rna_cbio_agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>coadread_rna_cbio_seq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>coadread_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"coadread_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"coadread_mut01_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://www.synapse.org/#!Synapse:syn1729383"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">224</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA colon &amp; rectum adenocarcinoma (COADREAD) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>coadread_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>coadread_mut01_ucsc_mutation</td>
<td>ucsc</td>
<td>mut01</td>
<td>224</td>
<td>TCGA colon &amp; rectum adenocarcinoma (COADREAD) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
</tbody></table>

<h2 id="lung-lung-adenocarcinoma-amp-lung-squamous-cell-carcinoma">LUNG - Lung adenocarcinoma &amp; Lung squamous cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lung_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lung_facs_demo_flow</td>
<td>molecular</td>
<td>demo</td>
<td>facs</td>
</tr>
<tr>
<td>lung_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>lung_mut01_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>lung_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>lung_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>lung_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>lung_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lung_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lung_rna_cbio_agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lung_rna_cbio_seq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lung_rna_cbio_u133</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lung_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>lung_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lung_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1016</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lung_mut01_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>lung_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>1016</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>lung_mut01_ucsc_mutation</td>
<td>ucsc</td>
<td>mut01</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="coad-colon-adenocarcinoma">COAD - Colon adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>coad_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_newtumor-followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="hg19-genome-platform">HG19 - Genome Platform</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>hg19_genesets_hgnc_import</td>
<td>category</td>
<td>hgnc</td>
<td>genesets</td>
</tr>
<tr>
<td>hg19_genesets_orghs_1e+05</td>
<td>category</td>
<td>orgHs</td>
<td>genesets</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>

<h2 id="laml-acute-myeloid-leukemia">LAML - Acute Myeloid Leukemia</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>laml_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>laml_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>laml_mut01_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>laml_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"laml_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LAML/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">191</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA acute myeloid leukemia (LAML) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"laml_mut01_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://www.synapse.org/#!Synapse:syn1729383"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">196</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript mutationMatrix processed on 2015-01-28"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA PANCAN strictly filtered maf files (file names: *_cleaned_filtered.maf) download from Synapse, processed into gene by sample matrix at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA acute myeloid leukemia (LAML) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>laml_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>191</td>
<td>TCGA acute myeloid leukemia (LAML) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>laml_mut01_ucsc_mutation</td>
<td>ucsc</td>
<td>mut01</td>
<td>196</td>
<td>TCGA acute myeloid leukemia (LAML) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="read-rectum-adenocarcinoma">READ - Rectum adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>read_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_newtumor-followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
<tr>
<td>read_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="ucs-uterine-carcinosarcoma">UCS - Uterine Carcinosarcoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>ucs_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ucs_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>ucs_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="uvm-uveal-melanoma">UVM - Uveal Melanoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>uvm_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>uvm_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>uvm_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="thym-thymoma">THYM - Thymoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>thym_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>thym_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>thym_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="tgct-testicular-germ-cell-tumors">TGCT - Testicular Germ Cell Tumors</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>tgct_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>tgct_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>tgct_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="pcpg-pheochromocytoma-and-paraganglioma">PCPG - Pheochromocytoma and Paraganglioma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>pcpg_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>pcpg_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>pcpg_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="ov-ovarian-serous-cystadenocarcinoma">OV - Ovarian serous cystadenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>ov_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_newtumor-followup_tcga_v1p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>ov_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>ov_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

<h2 id="meso-mesothelioma">MESO - Mesothelioma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>meso_drug_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_newtumor-followup_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_othermalignancy_tcga_v4p0</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_patient_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_radiation_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>meso_rna_ucsc_hiseq</td>
<td>molecular</td>
<td>ucsc</td>
<td>rna</td>
</tr>
<tr>
<td>meso_protein_ucsc_rppa</td>
<td>molecular</td>
<td>ucsc</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight plaintext"><code>
false
</code></pre>
<pre class="highlight plaintext"><code>
false
</code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
<tr>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
<td>undefined</td>
</tr>
</tbody></table>

      </div>
      <div class="dark-box">
          <div class="lang-selector">
                <a href="#" data-language-name="python">python</a>
                <a href="#" data-language-name="javascript">javascript</a>
                <a href="#" data-language-name="r">r</a>
          </div>
      </div>
    </div>
  </body>
</html>
