<!doctype html>
<html>
  <head>
    <meta charset="utf-8">
    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
    <title>API Reference</title>

    <style>
      .highlight table td { padding: 5px; }
.highlight table pre { margin: 0; }
.highlight, .highlight .w {
  color: #f8f8f2;
  background-color: #272822;
}
.highlight .err {
  color: #151515;
  background-color: #ac4142;
}
.highlight .c, .highlight .cd, .highlight .cm, .highlight .c1, .highlight .cs {
  color: #505050;
}
.highlight .cp {
  color: #f4bf75;
}
.highlight .nt {
  color: #f4bf75;
}
.highlight .o, .highlight .ow {
  color: #d0d0d0;
}
.highlight .p, .highlight .pi {
  color: #d0d0d0;
}
.highlight .gi {
  color: #90a959;
}
.highlight .gd {
  color: #ac4142;
}
.highlight .gh {
  color: #6a9fb5;
  background-color: #151515;
  font-weight: bold;
}
.highlight .k, .highlight .kn, .highlight .kp, .highlight .kr, .highlight .kv {
  color: #aa759f;
}
.highlight .kc {
  color: #d28445;
}
.highlight .kt {
  color: #d28445;
}
.highlight .kd {
  color: #d28445;
}
.highlight .s, .highlight .sb, .highlight .sc, .highlight .sd, .highlight .s2, .highlight .sh, .highlight .sx, .highlight .s1 {
  color: #90a959;
}
.highlight .sr {
  color: #75b5aa;
}
.highlight .si {
  color: #8f5536;
}
.highlight .se {
  color: #8f5536;
}
.highlight .nn {
  color: #f4bf75;
}
.highlight .nc {
  color: #f4bf75;
}
.highlight .no {
  color: #f4bf75;
}
.highlight .na {
  color: #6a9fb5;
}
.highlight .m, .highlight .mf, .highlight .mh, .highlight .mi, .highlight .il, .highlight .mo, .highlight .mb, .highlight .mx {
  color: #90a959;
}
.highlight .ss {
  color: #90a959;
}
    </style>
    <link href="stylesheets/screen.css" rel="stylesheet" media="screen" />
    <link href="stylesheets/print.css" rel="stylesheet" media="print" />
      <script src="javascripts/all.js"></script>
  </head>

  <body class="index" data-languages="[&quot;python&quot;,&quot;javascript&quot;,&quot;r&quot;]">
    <a href="#" id="nav-button">
      <span>
        NAV
        <img src="images/navbar.png" />
      </span>
    </a>
    <div class="tocify-wrapper">
      <img src="images/logo.png" />
        <div class="lang-selector">
              <a href="#" data-language-name="python">python</a>
              <a href="#" data-language-name="javascript">javascript</a>
              <a href="#" data-language-name="r">r</a>
        </div>
        <div class="search">
          <input type="text" class="search" id="input-search" placeholder="Search">
        </div>
        <ul class="search-results"></ul>
      <div id="toc">
      </div>
        <ul class="toc-footer">
            <li>email to contact@oncoscape.org</li>
            <li><a href='https://github.com/FredHutch/Oncoscape'>Oncoscape Github Site</a></li>
        </ul>
    </div>
    <div class="page-wrapper">
      <div class="dark-box"></div>
      <div class="content">
        <h1 id="introduction">Introduction</h1>

<p><a href="https://oncoscape.sttrcancer.org/#/">Oncoscape</a> is a web application that hosts an integrated suite of analysis tools for users to explore hypotheses related to molecular and clinical data in order to better understand cancer biology and treatment options. Oncoscape is as an SPA &ndash; a single page web application &ndash; using JavaScript in the browser and R (primarily) on the backend server for statistical calculations. For more detailed information, please read <a href="https://github.com/FredHutch/Oncoscape">wiki</a>.</p>

<h1 id="authentication">Authentication</h1>

<p>Oncoscape API authentication occurs at Mongo DB level. </p>

<h1 id="rest-api-queries">Rest API Queries</h1>

          <h2 id="example-to-access-one-collection-from-browser">Example to access one collection from browser</h2>

<h3 id="http-request">HTTP Request</h3>

<p><code class="prettyprint">GET http://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical/</code></p>

<blockquote>
<p>Here we only show the first record in gbm_patient_tcga_clinical</p>
</blockquote>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"_id"</span><span class="p">:</span><span class="w"> </span><span class="s2">"57bce8ec1debbd62f4bf3501"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"patient_ID"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA-02-0001-01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"history_lgg_dx_of_brain_tissue"</span><span class="p">:</span><span class="w"> </span><span class="kc">false</span><span class="p">,</span><span class="w">
    </span><span class="nt">"prospective_collection"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"retrospective_collection"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"gender"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FEMALE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_birth"</span><span class="p">:</span><span class="w"> </span><span class="mi">-16179</span><span class="p">,</span><span class="w">
    </span><span class="nt">"race"</span><span class="p">:</span><span class="w"> </span><span class="s2">"WHITE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"ethnicity"</span><span class="p">:</span><span class="w"> </span><span class="s2">"NOT HISPANIC OR LATINO"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"history_other_malignancy"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"history_neoadjuvant_treatment"</span><span class="p">:</span><span class="w"> </span><span class="kc">true</span><span class="p">,</span><span class="w">
    </span><span class="nt">"diagnosis_year"</span><span class="p">:</span><span class="w"> </span><span class="mi">1009872000</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pathologic_method"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status_vital"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DEAD"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_last_contact"</span><span class="p">:</span><span class="w"> </span><span class="mi">279</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_death"</span><span class="p">:</span><span class="w"> </span><span class="mi">358</span><span class="p">,</span><span class="w">
    </span><span class="nt">"status_tumor"</span><span class="p">:</span><span class="w"> </span><span class="s2">"WITH TUMOR"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"KPS"</span><span class="p">:</span><span class="w"> </span><span class="mi">80</span><span class="p">,</span><span class="w">
    </span><span class="nt">"ECOG"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"encounter_type"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"radiation_treatment_adjuvant"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"pharmaceutical_tx_adjuvant"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"treatment_outcome_first_course"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"new_tumor_event_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"age_at_initial_pathologic_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="mi">44</span><span class="p">,</span><span class="w">
    </span><span class="nt">"anatomic_organ_subdivision"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"days_to_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
    </span><span class="nt">"disease_code"</span><span class="p">:</span><span class="w"> </span><span class="kc">null</span><span class="p">,</span><span class="w">
    </span><span class="nt">"histologic_diagnosis"</span><span class="p">:</span><span class="w"> </span><span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"icd_10"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C71.9"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"icd_3_histology"</span><span class="p">:</span><span class="w"> </span><span class="s2">"9440/3"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"icd_3"</span><span class="p">:</span><span class="w"> </span><span class="s2">"C71.9"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"tissue_source_site_code"</span><span class="p">:</span><span class="w"> </span><span class="s2">"02"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"tumor_tissue_site"</span><span class="p">:</span><span class="w"> </span><span class="s2">"BRAIN"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<h2 id="query-collection-from-browser">Query Collection from Browser</h2>

<h3 id="http-request">HTTP Request</h3>

<blockquote>
<p>Here we show the first two records that meet the below criteria: gender is male, race is white. We have skipped the first five records from the results. And we only show three fields (patient_ID, gender and race.</p>
</blockquote>
<pre class="highlight plaintext"><code>
[
    {
        "_id": "57bce8ec1debbd62f4bf350e",
        "patient_ID": "TCGA-02-0024-01",
        "gender": "MALE",
        "race": "WHITE",
        "histologic_diagnosis": "GLIOBLASTOMA MULTIFORME TREATED PRIMARY"
    },
    {
        "_id": "57bce8ec1debbd62f4bf350f",
        "patient_ID": "TCGA-02-0025-01",
        "gender": "MALE",
        "race": "WHITE",
        "histologic_diagnosis": "GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"
    }
]
</code></pre>

<p>Filter by gender and race and only show the selected fields</p>

<p><code class="prettyprint">GET http://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical/?q={&quot;gender&quot;:&quot;MALE&quot;, &quot;race&quot;:&quot;WHITE&quot;,&quot;$fields&quot;:[&quot;gender&quot;,&quot;race&quot;,&quot;patient_ID&quot;],&quot;$skip&quot;:5,&quot;$limit&quot;:2}</code></p>

<p>only show gender, race and patient_ID</p>

<p><code class="prettyprint">&quot;$fields&quot;:[&quot;gender&quot;,&quot;race&quot;,&quot;patient_ID&quot;]</code></p>

<p>skip the first five records</p>

<p><code class="prettyprint">&quot;$skip&quot;:5</code></p>

<p>limit the final output to two records.</p>

<p><code class="prettyprint">&quot;$limit&quot;:2</code></p>

<h2 id="fetch-json-formatted-data-using-programming-languages">Fetch JSON-Formatted Data Using Programming Languages</h2>

<blockquote>
<p>Fetch JSON formatted data using R, Python, or javascript</p>
</blockquote>
<pre class="highlight javascript"><code><span class="kd">var</span> <span class="nx">collection</span> <span class="o">=</span> <span class="s2">"gbm_patient_tcga_clinical"</span><span class="p">;</span>
<span class="kd">var</span> <span class="nx">url</span> <span class="o">=</span> <span class="s2">"https://dev.oncoscape.sttrcancer.io/api/"</span> <span class="o">+</span> <span class="nx">collection</span> <span class="o">+</span> <span class="s2">"/?q="</span><span class="p">;</span>
<span class="nx">$</span><span class="p">.</span><span class="nx">get</span><span class="p">(</span><span class="nx">url</span><span class="p">,</span> <span class="kd">function</span><span class="p">(</span><span class="nx">data</span><span class="p">)</span> <span class="p">{</span>
     <span class="kd">var</span> <span class="nx">field_names</span> <span class="o">=</span> <span class="nb">Object</span><span class="p">.</span><span class="nx">keys</span><span class="p">(</span><span class="nx">data</span><span class="p">[</span><span class="mi">0</span><span class="p">]);</span>
     <span class="kd">var</span> <span class="nx">count</span> <span class="o">=</span> <span class="nx">data</span><span class="p">.</span><span class="nx">length</span><span class="p">;</span>
     <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="s2">"fields name of the first records: "</span> <span class="o">+</span> <span class="nx">field_names</span><span class="p">);</span>
     <span class="nx">console</span><span class="p">.</span><span class="nx">log</span><span class="p">(</span><span class="s2">"counts: "</span> <span class="o">+</span> <span class="nx">count</span><span class="p">);</span>
  <span class="p">});</span>
</code></pre>
<pre class="highlight r"><code><span class="n">install.packages</span><span class="p">(</span><span class="s2">"jsonlite"</span><span class="p">)</span>
<span class="n">install.packages</span><span class="p">(</span><span class="s2">"curl"</span><span class="p">)</span>
<span class="n">library</span><span class="p">(</span><span class="n">jsonlite</span><span class="p">)</span>
<span class="n">library</span><span class="p">(</span><span class="n">curl</span><span class="p">)</span>
<span class="n">gbm_patient</span> <span class="o">&lt;-</span> <span class="n">fromJSON</span><span class="p">(</span><span class="s2">"https://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical"</span><span class="p">)</span>
<span class="n">str</span><span class="p">(</span><span class="n">gbm_patient</span><span class="p">,</span> <span class="n">max.level</span><span class="o">=</span><span class="m">2</span><span class="p">)</span>
<span class="s1">'data.frame'</span><span class="o">:</span> <span class="m">596</span> <span class="n">obs.</span> <span class="n">of</span>  <span class="m">33</span> <span class="n">variables</span><span class="o">:</span>
<span class="o">$</span> <span class="n">patient_ID</span>                         <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"TCGA-02-0001-01"</span> <span class="s2">"TCGA-02-0003-01"</span> <span class="s2">"TCGA-02-0004-01"</span> <span class="s2">"TCGA-02-0006-01"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">history_lgg_dx_of_brain_tissue</span>     <span class="o">:</span> <span class="n">logi</span>  <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">prospective_collection</span>             <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">retrospective_collection</span>           <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">gender</span>                             <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"FEMALE"</span> <span class="s2">"MALE"</span> <span class="s2">"MALE"</span> <span class="s2">"FEMALE"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_birth</span>                      <span class="o">:</span> <span class="n">int</span>  <span class="m">-16179</span> <span class="m">-18341</span> <span class="m">-21617</span> <span class="m">-20516</span> <span class="m">-14806</span> <span class="m">-22457</span> <span class="m">-7452</span> <span class="m">-6926</span> <span class="m">-9369</span> <span class="m">-18404</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">race</span>                               <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"WHITE"</span> <span class="s2">"WHITE"</span> <span class="s2">"WHITE"</span> <span class="s2">"WHITE"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">ethnicity</span>                          <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="s2">"NOT HISPANIC OR LATINO"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">history_other_malignancy</span>           <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">history_neoadjuvant_treatment</span>      <span class="o">:</span> <span class="n">logi</span>  <span class="n">TRUE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">FALSE</span> <span class="n">TRUE</span> <span class="n">FALSE</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">diagnosis_year</span>                     <span class="o">:</span> <span class="n">int</span>  <span class="m">1009872000</span> <span class="m">1041408000</span> <span class="m">1009872000</span> <span class="m">1009872000</span> <span class="m">1009872000</span> <span class="m">1041408000</span> <span class="m">1009872000</span> <span class="m">1072944000</span> <span class="m">852105600</span> <span class="m">1009872000</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">pathologic_method</span>                  <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">pathologic_method</span>                  <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">status_vital</span>                       <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"DEAD"</span> <span class="s2">"DEAD"</span> <span class="s2">"DEAD"</span> <span class="s2">"DEAD"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_last_contact</span>               <span class="o">:</span> <span class="n">int</span>  <span class="m">279</span> <span class="m">144</span> <span class="m">345</span> <span class="m">558</span> <span class="m">705</span> <span class="m">322</span> <span class="m">1077</span> <span class="m">630</span> <span class="m">2512</span> <span class="m">627</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_death</span>                      <span class="o">:</span> <span class="n">int</span>  <span class="m">358</span> <span class="m">144</span> <span class="m">345</span> <span class="m">558</span> <span class="m">705</span> <span class="m">322</span> <span class="m">1077</span> <span class="m">630</span> <span class="m">2512</span> <span class="m">627</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">status_tumor</span>                       <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"WITH TUMOR"</span> <span class="s2">"WITH TUMOR"</span> <span class="s2">"WITH TUMOR"</span> <span class="s2">"WITH TUMOR"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">KPS</span>                                <span class="o">:</span> <span class="n">int</span>  <span class="m">80</span> <span class="m">100</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">80</span> <span class="m">100</span> <span class="m">80</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">ECOG</span>                               <span class="o">:</span> <span class="n">int</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">encounter_type</span>                     <span class="o">:</span> <span class="n">chr</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">radiation_treatment_adjuvant</span>       <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">pharmaceutical_tx_adjuvant</span>         <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">treatment_outcome_first_course</span>     <span class="o">:</span> <span class="n">chr</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">new_tumor_event_diagnosis</span>          <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">age_at_initial_pathologic_diagnosis</span><span class="o">:</span> <span class="n">int</span>  <span class="m">44</span> <span class="m">50</span> <span class="m">59</span> <span class="m">56</span> <span class="m">40</span> <span class="m">61</span> <span class="m">20</span> <span class="m">18</span> <span class="m">25</span> <span class="m">50</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">anatomic_organ_subdivision</span>         <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">days_to_diagnosis</span>                  <span class="o">:</span> <span class="n">int</span>  <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="m">0</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">disease_code</span>                       <span class="o">:</span> <span class="n">logi</span>  <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="n">NA</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">histologic_diagnosis</span>               <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="s2">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">icd_10</span>                             <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">icd_3_histology</span>                    <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"9440/3"</span> <span class="s2">"9440/3"</span> <span class="s2">"9440/3"</span> <span class="s2">"9440/3"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">icd_3</span>                              <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="s2">"C71.9"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">tissue_source_site_code</span>            <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"02"</span> <span class="s2">"02"</span> <span class="s2">"02"</span> <span class="s2">"02"</span> <span class="k">...</span>
<span class="o">$</span> <span class="n">tumor_tissue_site</span>                  <span class="o">:</span> <span class="n">chr</span>  <span class="s2">"BRAIN"</span> <span class="s2">"BRAIN"</span> <span class="s2">"BRAIN"</span> <span class="s2">"BRAIN"</span> <span class="k">...</span>
</code></pre>
<pre class="highlight python"><code>
<span class="o">&gt;</span> <span class="n">shell</span> <span class="n">commands</span><span class="p">:</span> <span class="n">sudo</span> <span class="n">pip</span> <span class="n">install</span> <span class="n">pymongo</span><span class="p">,</span> <span class="n">simplejson</span><span class="p">,</span> <span class="n">urllib2</span><span class="p">,</span> <span class="n">json</span>

<span class="kn">import</span> <span class="nn">urllib2</span>
<span class="kn">import</span> <span class="nn">json</span>
<span class="kn">import</span> <span class="nn">simplejson</span>
<span class="n">url</span> <span class="o">=</span> <span class="s">"https://dev.oncoscape.sttrcancer.io/api/gbm_patient_tcga_clinical"</span>
<span class="n">response</span> <span class="o">=</span> <span class="n">urlli2</span><span class="o">.</span><span class="n">urlopen</span><span class="p">(</span><span class="n">url</span><span class="p">)</span>
<span class="n">data</span> <span class="o">=</span> <span class="n">simplejson</span><span class="o">.</span><span class="n">load</span><span class="p">(</span><span class="n">response</span><span class="p">)</span>
<span class="k">print</span> <span class="n">json</span><span class="o">.</span><span class="n">dumps</span><span class="p">(</span><span class="n">data</span><span class="p">[</span><span class="mi">0</span><span class="p">:</span><span class="mi">2</span><span class="p">],</span> <span class="n">indent</span><span class="o">=</span><span class="mi">4</span><span class="p">,</span> <span class="n">sort_keys</span><span class="o">=</span><span class="bp">True</span><span class="p">)</span>

<span class="p">[</span>
    <span class="p">{</span>
        <span class="s">"ECOG"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"KPS"</span><span class="p">:</span> <span class="mi">80</span><span class="p">,</span>
        <span class="s">"age_at_initial_pathologic_diagnosis"</span><span class="p">:</span> <span class="mi">44</span><span class="p">,</span>
        <span class="s">"anatomic_organ_subdivision"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"days_to_birth"</span><span class="p">:</span> <span class="o">-</span><span class="mi">16179</span><span class="p">,</span>
        <span class="s">"days_to_death"</span><span class="p">:</span> <span class="mi">358</span><span class="p">,</span>
        <span class="s">"days_to_diagnosis"</span><span class="p">:</span> <span class="mi">0</span><span class="p">,</span>
        <span class="s">"days_to_last_contact"</span><span class="p">:</span> <span class="mi">279</span><span class="p">,</span>
        <span class="s">"diagnosis_year"</span><span class="p">:</span> <span class="mi">1009872000</span><span class="p">,</span>
        <span class="s">"disease_code"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"encounter_type"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"ethnicity"</span><span class="p">:</span> <span class="s">"NOT HISPANIC OR LATINO"</span><span class="p">,</span>
        <span class="s">"gender"</span><span class="p">:</span> <span class="s">"FEMALE"</span><span class="p">,</span>
        <span class="s">"histologic_diagnosis"</span><span class="p">:</span> <span class="s">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span><span class="p">,</span>
        <span class="s">"history_lgg_dx_of_brain_tissue"</span><span class="p">:</span> <span class="n">false</span><span class="p">,</span>
        <span class="s">"history_neoadjuvant_treatment"</span><span class="p">:</span> <span class="n">true</span><span class="p">,</span>
        <span class="s">"history_other_malignancy"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"icd_10"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3_histology"</span><span class="p">:</span> <span class="s">"9440/3"</span><span class="p">,</span>
        <span class="s">"new_tumor_event_diagnosis"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"pathologic_method"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"patient_ID"</span><span class="p">:</span> <span class="s">"TCGA-02-0001-01"</span><span class="p">,</span>
        <span class="s">"pharmaceutical_tx_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"prospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"race"</span><span class="p">:</span> <span class="s">"WHITE"</span><span class="p">,</span>
        <span class="s">"radiation_treatment_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"retrospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"status_tumor"</span><span class="p">:</span> <span class="s">"WITH TUMOR"</span><span class="p">,</span>
        <span class="s">"status_vital"</span><span class="p">:</span> <span class="s">"DEAD"</span><span class="p">,</span>
        <span class="s">"tissue_source_site_code"</span><span class="p">:</span> <span class="s">"02"</span><span class="p">,</span>
        <span class="s">"treatment_outcome_first_course"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"tumor_tissue_site"</span><span class="p">:</span> <span class="s">"BRAIN"</span>
    <span class="p">},</span>
    <span class="p">{</span>
        <span class="s">"ECOG"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"KPS"</span><span class="p">:</span> <span class="mi">100</span><span class="p">,</span>
        <span class="s">"age_at_initial_pathologic_diagnosis"</span><span class="p">:</span> <span class="mi">50</span><span class="p">,</span>
        <span class="s">"anatomic_organ_subdivision"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"days_to_birth"</span><span class="p">:</span> <span class="o">-</span><span class="mi">18341</span><span class="p">,</span>
        <span class="s">"days_to_death"</span><span class="p">:</span> <span class="mi">144</span><span class="p">,</span>
        <span class="s">"days_to_diagnosis"</span><span class="p">:</span> <span class="mi">0</span><span class="p">,</span>
        <span class="s">"days_to_last_contact"</span><span class="p">:</span> <span class="mi">144</span><span class="p">,</span>
        <span class="s">"diagnosis_year"</span><span class="p">:</span> <span class="mi">1041408000</span><span class="p">,</span>
        <span class="s">"disease_code"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"encounter_type"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"ethnicity"</span><span class="p">:</span> <span class="s">"NOT HISPANIC OR LATINO"</span><span class="p">,</span>
        <span class="s">"gender"</span><span class="p">:</span> <span class="s">"MALE"</span><span class="p">,</span>
        <span class="s">"histologic_diagnosis"</span><span class="p">:</span> <span class="s">"GLIOBLASTOMA MULTIFORME UNTREATED PRIMARY"</span><span class="p">,</span>
        <span class="s">"history_lgg_dx_of_brain_tissue"</span><span class="p">:</span> <span class="n">false</span><span class="p">,</span>
        <span class="s">"history_neoadjuvant_treatment"</span><span class="p">:</span> <span class="n">false</span><span class="p">,</span>
        <span class="s">"history_other_malignancy"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"icd_10"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3"</span><span class="p">:</span> <span class="s">"C71.9"</span><span class="p">,</span>
        <span class="s">"icd_3_histology"</span><span class="p">:</span> <span class="s">"9440/3"</span><span class="p">,</span>
        <span class="s">"new_tumor_event_diagnosis"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"pathologic_method"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"patient_ID"</span><span class="p">:</span> <span class="s">"TCGA-02-0003-01"</span><span class="p">,</span>
        <span class="s">"pharmaceutical_tx_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"prospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"race"</span><span class="p">:</span> <span class="s">"WHITE"</span><span class="p">,</span>
        <span class="s">"radiation_treatment_adjuvant"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"retrospective_collection"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"status_tumor"</span><span class="p">:</span> <span class="s">"WITH TUMOR"</span><span class="p">,</span>
        <span class="s">"status_vital"</span><span class="p">:</span> <span class="s">"DEAD"</span><span class="p">,</span>
        <span class="s">"tissue_source_site_code"</span><span class="p">:</span> <span class="s">"02"</span><span class="p">,</span>
        <span class="s">"treatment_outcome_first_course"</span><span class="p">:</span> <span class="n">null</span><span class="p">,</span>
        <span class="s">"tumor_tissue_site"</span><span class="p">:</span> <span class="s">"BRAIN"</span>
    <span class="p">}</span>
<span class="p">]</span>
</code></pre>

<p>Users can access json-formatted data using URL link.</p>

<p>Here we show the example to access one collection using four different languages.</p>

<h1 id="collections-by-disease">Collections by Disease</h1>

<h2 id="hg19-genome-platform">HG19 - Genome Platform</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>hg19_genesets_hgnc_import</td>
<td>category</td>
<td>hgnc</td>
<td>genesets</td>
</tr>
<tr>
<td>hg19_genesets_orghs_10000</td>
<td>category</td>
<td>orgHs</td>
<td>genesets</td>
</tr>
<tr>
<td>hg19_genesets_orghs_1e+05</td>
<td>category</td>
<td>orgHs</td>
<td>genesets</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>

<h2 id="brca-breast-invasive-carcinoma">BRCA - Breast invasive carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>brca_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_patient_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_drug_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_radiation_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_followup-v1p5_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_followup-v2p1_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_followup-v4p0_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_newtumor-followup-v4p0_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_othermalignancy-v4p0_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_events_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brca_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>brca_mut_cbio_wxs</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>brca_mut01_cbio_import</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>brca_rna_cbio_microarray-agilent-median-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>brca_rna_cbio_rnaseq-median-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>brca_protein_cbio_rppa-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
<tr>
<td>brca_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>brca_mut_ucsc_mutationcuratedwustlgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>brca_mut01_ucsc_import</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>brca_psi_bradleylab_miso</td>
<td>molecular</td>
<td>bradleyLab</td>
<td>psi</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1079</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 1079 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_mut_cbio_wxs"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">980</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_mut01_cbio_import"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">980</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_methylation_cbio_hm27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">315</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27) beta-values for genes in 342 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_methylation_cbio_hm450"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">737</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450) beta-values for genes in 833 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_rna_cbio_microarray-agilent-median-zscore"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1098</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (microarray)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_rna_cbio_rnaseq-median-zscore"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">490</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (RNA Seq V2 RSEM)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_protein_cbio_rppa-zscore"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">409</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/BRCA/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1079</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA breast invasive carcinoma (BRCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_mut_ucsc_mutationcuratedwustlgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/brca/gsc/genome.wustl.edu/illuminaga_dnaseq_curated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">982</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA breast invasive carcinoma (BRCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Genome Institute at Washington University Sequencing Center using the WashU pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brca_mut01_ucsc_import"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/brca/gsc/genome.wustl.edu/illuminaga_dnaseq_curated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">982</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA breast invasive carcinoma (BRCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Genome Institute at Washington University Sequencing Center using the WashU pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>brca_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>1079</td>
<td>Putative copy-number calls on 1079 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>brca_mut_cbio_wxs</td>
<td>cbio</td>
<td>mut</td>
<td>980</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>brca_mut01_cbio_import</td>
<td>cbio</td>
<td>mut01</td>
<td>980</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm27</td>
<td>cbio</td>
<td>methylation</td>
<td>315</td>
<td>Methylation (HM27) beta-values for genes in 342 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>brca_methylation_cbio_hm450</td>
<td>cbio</td>
<td>methylation</td>
<td>737</td>
<td>Methylation (HM450) beta-values for genes in 833 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>brca_rna_cbio_microarray-agilent-median-zscore</td>
<td>cbio</td>
<td>rna</td>
<td>1098</td>
<td>mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>brca_rna_cbio_rnaseq-median-zscore</td>
<td>cbio</td>
<td>rna</td>
<td>490</td>
<td>mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>brca_protein_cbio_rppa-zscore</td>
<td>cbio</td>
<td>protein</td>
<td>409</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
<tr>
<td>brca_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>1079</td>
<td>TCGA breast invasive carcinoma (BRCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>brca_mut_ucsc_mutationcuratedwustlgene</td>
<td>ucsc</td>
<td>mut</td>
<td>982</td>
<td>TCGA breast invasive carcinoma (BRCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Genome Institute at Washington University Sequencing Center using the WashU pipeline method.</td>
</tr>
<tr>
<td>brca_mut01_ucsc_import</td>
<td>ucsc</td>
<td>mut01</td>
<td>982</td>
<td>TCGA breast invasive carcinoma (BRCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Genome Institute at Washington University Sequencing Center using the WashU pipeline method.</td>
</tr>
</tbody></table>

<h2 id="esca-esophageal-carcinoma">ESCA - Esophageal carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>esca_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"esca_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/ESCA/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">184</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA esophageal carcinoma (ESCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>esca_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>184</td>
<td>TCGA esophageal carcinoma (ESCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
</tbody></table>

<h2 id="lung-lung-adenocarcinoma-amp-lung-squamous-cell-carcinoma">LUNG - Lung adenocarcinoma &amp; Lung squamous cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lung_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lung_facs_demo_flow</td>
<td>molecular</td>
<td>demo</td>
<td>facs</td>
</tr>
<tr>
<td>lung_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>lung_mut_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>lung_mut01_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>lung_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>lung_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>lung_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>lung_methylation_cbio_methylationhm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lung_methylation_cbio_methylationhm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lung_rna_cbio_microarray-agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lung_rna_cbio_rnaseq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lung_rna_cbio_microarray-u133</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lung_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lung_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1016</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lung_mut_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lung_mut01_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>lung_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>1016</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>lung_mut_ucsc_mutation</td>
<td>ucsc</td>
<td>mut</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>lung_mut01_ucsc_mutation</td>
<td>ucsc</td>
<td>mut01</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
</tbody></table>

<h2 id="coad-colon-adenocarcinoma">COAD - Colon adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>coad_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coad_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>

<h2 id="gbm-glioblastoma-multiforme">GBM - Glioblastoma multiforme</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>gbm_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>gbm_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>gbm_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>gbm_mut01_ucsc_import</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gbm_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/GBM/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">577</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA glioblastoma multiforme (GBM) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gbm_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/gbm/gsc/broad.mit.edu/illuminaga_dnaseq/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">291</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA glioblastoma multiforme (GBM) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"gbm_mut01_ucsc_import"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/gbm/gsc/broad.mit.edu/illuminaga_dnaseq/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">291</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA glioblastoma multiforme (GBM) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>gbm_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>577</td>
<td>TCGA glioblastoma multiforme (GBM) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>gbm_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>291</td>
<td>TCGA glioblastoma multiforme (GBM) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>gbm_mut01_ucsc_import</td>
<td>ucsc</td>
<td>mut01</td>
<td>291</td>
<td>TCGA glioblastoma multiforme (GBM) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
</tbody></table>

<h2 id="coadread-colon-adenocarcinoma-amp-rectum-adenocarcinoma">COADREAD - Colon adenocarcinoma &amp; Rectum adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>coadread_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>coadread_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>coadread_mut_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>coadread_mut01_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>coadread_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>coadread_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>coadread_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>coadread_methylation_cbio_methylationhm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>coadread_methylation_cbio_methylationhm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>coadread_rna_cbio_microarray-agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>coadread_rna_cbio_rnaseq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>coadread_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"coadread_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"coadread_mut_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://www.synapse.org/#!Synapse:syn1729383"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">224</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA colon &amp; rectum adenocarcinoma (COADREAD) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"coadread_mut01_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://www.synapse.org/#!Synapse:syn1729383"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">224</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA colon &amp; rectum adenocarcinoma (COADREAD) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>coadread_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>coadread_mut_ucsc_mutation</td>
<td>ucsc</td>
<td>mut</td>
<td>224</td>
<td>TCGA colon &amp; rectum adenocarcinoma (COADREAD) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>coadread_mut01_ucsc_mutation</td>
<td>ucsc</td>
<td>mut01</td>
<td>224</td>
<td>TCGA colon &amp; rectum adenocarcinoma (COADREAD) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
</tbody></table>

<h2 id="hnsc-head-and-neck-squamous-cell-carcinoma">HNSC - Head and Neck squamous cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>hnsc_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_patient_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_drug_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_radiation_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_followup-v4p8_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_newtumor_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_newtumor-followup-v4p8_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_events_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>hnsc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>hnsc_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>hnsc_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>hnsc_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>hnsc_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>hnsc_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>hnsc_methylation_cbio_methylationhm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>hnsc_rna_cbio_rnaseq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>hnsc_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/HNSC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">522</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA head &amp; neck squamous cell carcinoma (HNSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/hnsc/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">509</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA head &amp; neck squamous cell carcinoma (HNSC) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/hnsc/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">509</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA head &amp; neck squamous cell carcinoma (HNSC) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">522</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 522 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">306</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">306</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_methylation_cbio_methylationhm450"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">530</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450) beta-values for genes in 580 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_rna_cbio_rnaseq"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">498</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA expression (RNA Seq V2 RSEM)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Expression levels for 20532 genes in 541 hnsc cases (RNA Seq V2 RSEM)."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"hnsc_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">212</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>hnsc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>522</td>
<td>TCGA head &amp; neck squamous cell carcinoma (HNSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>hnsc_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>509</td>
<td>TCGA head &amp; neck squamous cell carcinoma (HNSC) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>hnsc_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>509</td>
<td>TCGA head &amp; neck squamous cell carcinoma (HNSC) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>hnsc_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>522</td>
<td>Putative copy-number calls on 522 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>hnsc_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>306</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>hnsc_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>306</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>hnsc_methylation_cbio_methylationhm450</td>
<td>cbio</td>
<td>methylation</td>
<td>530</td>
<td>Methylation (HM450) beta-values for genes in 580 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>hnsc_rna_cbio_rnaseq</td>
<td>cbio</td>
<td>rna</td>
<td>498</td>
<td>Expression levels for 20532 genes in 541 hnsc cases (RNA Seq V2 RSEM).</td>
</tr>
<tr>
<td>hnsc_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>212</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
</tbody></table>

<h2 id="lgg-brain-lower-grade-glioma">LGG - Brain Lower Grade Glioma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lgg_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>lgg_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>lgg_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>lgg_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lgg_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LGG/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">513</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA brain lower grade glioma (LGG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lgg_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lgg_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>lgg_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>513</td>
<td>TCGA brain lower grade glioma (LGG) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>lgg_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>lgg_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
</tbody></table>

<h2 id="luad-lung-adenocarcinoma">LUAD - Lung adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>luad_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_events_tcga_clinical</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>luad_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>luad_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>luad_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>luad_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>luad_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>luad_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>luad_methylation_cbio_methylationhm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>luad_methylation_cbio_methylationhm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>luad_rna_cbio_microarray-agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>luad_rna_cbio_rnaseq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>luad_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LUAD/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">515</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung adenocarcinoma (LUAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/luad/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">543</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung adenocarcinoma (LUAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/luad/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">543</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung adenocarcinoma (LUAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">515</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 515 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">229</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">229</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_methylation_cbio_methylationhm27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">126</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27) beta-values for genes in 150 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_methylation_cbio_methylationhm450"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">451</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450) beta-values for genes in 483 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_rna_cbio_microarray-agilent"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA expression (microarray)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Expression levels for 17215 genes in 32 luad cases (Agilent microarray)."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_rna_cbio_rnaseq"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">490</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (RNA Seq V2 RSEM)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"luad_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">181</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>luad_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>515</td>
<td>TCGA lung adenocarcinoma (LUAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>luad_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>543</td>
<td>TCGA lung adenocarcinoma (LUAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>luad_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>543</td>
<td>TCGA lung adenocarcinoma (LUAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>luad_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>515</td>
<td>Putative copy-number calls on 515 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>luad_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>229</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>luad_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>229</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>luad_methylation_cbio_methylationhm27</td>
<td>cbio</td>
<td>methylation</td>
<td>126</td>
<td>Methylation (HM27) beta-values for genes in 150 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>luad_methylation_cbio_methylationhm450</td>
<td>cbio</td>
<td>methylation</td>
<td>451</td>
<td>Methylation (HM450) beta-values for genes in 483 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>luad_rna_cbio_microarray-agilent</td>
<td>cbio</td>
<td>rna</td>
<td>0</td>
<td>Expression levels for 17215 genes in 32 luad cases (Agilent microarray).</td>
</tr>
<tr>
<td>luad_rna_cbio_rnaseq</td>
<td>cbio</td>
<td>rna</td>
<td>490</td>
<td>mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>luad_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>181</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
</tbody></table>

<h2 id="lusc-lung-squamous-cell-carcinoma">LUSC - Lung squamous cell carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>lusc_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_patient_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_drug_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_radiation_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_newtumor_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>lusc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>lusc_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>lusc_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>lusc_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>lusc_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>lusc_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>lusc_methylation_cbio_methylationhm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lusc_methylation_cbio_methylationhm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>lusc_rna_cbio_microarray-agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_rna_cbio_rnaseq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_rna_cbio_microarray-u133</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>lusc_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LUSC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">501</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lung squamous cell carcinoma (LUSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">32</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 501 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">178</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">178</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_methylation_cbio_methylationhm27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">133</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM27) beta-values for genes in 160 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_methylation_cbio_methylationhm450"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">359</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450) beta-values for genes in 401 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_rna_cbio_microarray-agilent"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">154</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (microarray)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_rna_cbio_rnaseq"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">501</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (RNA Seq V2 RSEM)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_rna_cbio_microarray-u133"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">133</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (U133 microarray only)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores  (U133 microarray only) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"lusc_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">0</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>lusc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>501</td>
<td>TCGA lung squamous cell carcinoma (LUSC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>lusc_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>lusc_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>408</td>
<td>The dataset is combined from TCGA lung squamous cell carcinoma and lung adenocarcinoma datasets. TCGA lung cancer (LUNG) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>lusc_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>32</td>
<td>Putative copy-number calls on 501 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>lusc_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>178</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>lusc_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>178</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>lusc_methylation_cbio_methylationhm27</td>
<td>cbio</td>
<td>methylation</td>
<td>133</td>
<td>Methylation (HM27) beta-values for genes in 160 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>lusc_methylation_cbio_methylationhm450</td>
<td>cbio</td>
<td>methylation</td>
<td>359</td>
<td>Methylation (HM450) beta-values for genes in 401 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>lusc_rna_cbio_microarray-agilent</td>
<td>cbio</td>
<td>rna</td>
<td>154</td>
<td>mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>lusc_rna_cbio_rnaseq</td>
<td>cbio</td>
<td>rna</td>
<td>501</td>
<td>mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>lusc_rna_cbio_microarray-u133</td>
<td>cbio</td>
<td>rna</td>
<td>133</td>
<td>mRNA z-Scores  (U133 microarray only) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>lusc_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>0</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
</tbody></table>

<h2 id="prad-prostate-adenocarcinoma">PRAD - Prostate adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>prad_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_patient_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_drug_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_radiation_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_newtumor_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>prad_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>prad_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>prad_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>prad_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>prad_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>prad_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>prad_methylation_cbio_methylationhm450</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>prad_rna_cbio_microarray-agilent</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>prad_protein_cbio_rppa</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/PRAD/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">492</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"   cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA prostate adenocarcinoma (PRAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/prad/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">425</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA prostate adenocarcinoma (PRAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/prad/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">425</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA prostate adenocarcinoma (PRAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">492</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 492 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">261</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">261</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"methylation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_methylation_cbio_methylationhm450"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">425</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"METHYLATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"CONTINUOUS"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Methylation (HM450) beta-values for genes in 475 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_rna_cbio_microarray-agilent"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">487</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (microarray)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"prad_protein_cbio_rppa"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">164</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"PROTEIN_ARRAY_PROTEIN_LEVEL"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"protein/phosphoprotein level (RPPA)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)"</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>prad_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>492</td>
<td>TCGA prostate adenocarcinoma (PRAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>prad_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>425</td>
<td>TCGA prostate adenocarcinoma (PRAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>prad_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>425</td>
<td>TCGA prostate adenocarcinoma (PRAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>prad_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>492</td>
<td>Putative copy-number calls on 492 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>prad_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>261</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>prad_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>261</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>prad_methylation_cbio_methylationhm450</td>
<td>cbio</td>
<td>methylation</td>
<td>425</td>
<td>Methylation (HM450) beta-values for genes in 475 cases. For genes with multiple methylation probes, the probe most anti-correlated with expression.</td>
</tr>
<tr>
<td>prad_rna_cbio_microarray-agilent</td>
<td>cbio</td>
<td>rna</td>
<td>487</td>
<td>mRNA z-Scores (Agilent microarray) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
<tr>
<td>prad_protein_cbio_rppa</td>
<td>cbio</td>
<td>protein</td>
<td>164</td>
<td>Protein or phosphoprotein level (Z-scores) measured by reverse phase protein array (RPPA)</td>
</tr>
</tbody></table>

<h2 id="paad-pancreatic-adenocarcinoma">PAAD - Pancreatic adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>paad_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_followup-v4p4_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_newtumor-followup-v4p4_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>paad_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>paad_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>paad_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>paad_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>paad_mut_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>paad_mut01_cbio_mut</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>paad_rna_cbio_rnaseq</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/PAAD/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">184</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA pancreatic adenocarcinoma (PAAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/paad/gsc/hgsc.bcm.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">131</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA pancreatic adenocarcinoma (PAAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/paad/gsc/hgsc.bcm.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">131</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA pancreatic adenocarcinoma (PAAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_cnv_cbio_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">195</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"COPY_NUMBER_ALTERATION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"DISCRETE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number alterations from GISTIC"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Putative copy-number calls on 184 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">91</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_mut01_cbio_mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">91</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MUTATION_EXTENDED"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MAF"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Mutations"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Binary mutation data from whole exome sequencing."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cbio"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"rna"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"paad_rna_cbio_rnaseq"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">179</span><span class="p">,</span><span class="w">
    </span><span class="nt">"GENETIC_ALTERATION_TYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"MRNA_EXPRESSION"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"DATATYPE"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Z-SCORE"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"NAME"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA Expression z-Scores (RNA Seq V2 RSEM)"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>paad_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>184</td>
<td>TCGA pancreatic adenocarcinoma (PAAD) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>paad_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>131</td>
<td>TCGA pancreatic adenocarcinoma (PAAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method.</td>
</tr>
<tr>
<td>paad_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>131</td>
<td>TCGA pancreatic adenocarcinoma (PAAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method.</td>
</tr>
<tr>
<td>paad_cnv_cbio_gistic</td>
<td>cbio</td>
<td>cnv</td>
<td>195</td>
<td>Putative copy-number calls on 184 cases determined using GISTIC 2.0. Values: -2 = homozygous deletion; -1 = hemizygous deletion; 0 = neutral / no change; 1 = gain; 2 = high level amplification.</td>
</tr>
<tr>
<td>paad_mut_cbio_mut</td>
<td>cbio</td>
<td>mut</td>
<td>91</td>
<td>Mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>paad_mut01_cbio_mut</td>
<td>cbio</td>
<td>mut01</td>
<td>91</td>
<td>Binary mutation data from whole exome sequencing.</td>
</tr>
<tr>
<td>paad_rna_cbio_rnaseq</td>
<td>cbio</td>
<td>rna</td>
<td>179</td>
<td>mRNA z-Scores (RNA Seq V2 RSEM) compared to the expression distribution of each gene tumors that are diploid for this gene.</td>
</tr>
</tbody></table>

<h2 id="acc-adrenocortical-carcinoma">ACC - Adrenocortical carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>acc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"acc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/ACC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">90</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA adrenocortical carcinoma (ACC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>acc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>90</td>
<td>TCGA adrenocortical carcinoma (ACC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
</tbody></table>

<h2 id="blca-bladder-urothelial-carcinoma">BLCA - Bladder Urothelial Carcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>blca_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_followup-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_newtumor-followup-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>blca_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>blca_mut_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>blca_mut01_ucsc_mutationbroadgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"blca_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/BLCA/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">408</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA bladder urothelial carcinoma (BLCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"blca_mut_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/blca/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA bladder urothelial carcinoma (BLCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"blca_mut01_ucsc_mutationbroadgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/blca/gsc/broad.mit.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">238</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA bladder urothelial carcinoma (BLCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>blca_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>408</td>
<td>TCGA bladder urothelial carcinoma (BLCA) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>blca_mut_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut</td>
<td>238</td>
<td>TCGA bladder urothelial carcinoma (BLCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
<tr>
<td>blca_mut01_ucsc_mutationbroadgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>238</td>
<td>TCGA bladder urothelial carcinoma (BLCA) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Broad Institute Genome Sequencing Center using the MutDect method.</td>
</tr>
</tbody></table>

<h2 id="cesc-cervical-squamous-cell-carcinoma-and-endocervical-adenocarcinoma">CESC - Cervical squamous cell carcinoma and endocervical adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>cesc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cesc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/CESC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">295</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>cesc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>295</td>
<td>TCGA cervical squamous cell carcinoma and endocervical adenocarcinoma (CESC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
</tbody></table>

<h2 id="chol-cholangiocarcinoma">CHOL - Cholangiocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>chol_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"chol_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/CHOL/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">36</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA cholangiocarcinoma (CHOL) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>chol_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>36</td>
<td>TCGA cholangiocarcinoma (CHOL) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
</tbody></table>

<h2 id="dlbc-lymphoid-neoplasm-diffuse-large-b-cell-lymphoma">DLBC - Lymphoid Neoplasm Diffuse Large B-cell Lymphoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>dlbc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"dlbc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/DLBC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">48</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA lymphoid neoplasm diffuse large B-cell lymphoma (DLBC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>dlbc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>48</td>
<td>TCGA lymphoid neoplasm diffuse large B-cell lymphoma (DLBC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
</tbody></table>

<h2 id="laml-acute-myeloid-leukemia">LAML - Acute Myeloid Leukemia</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>laml_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>laml_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>laml_mut_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>laml_mut01_ucsc_mutation</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"laml_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/LAML/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">191</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA acute myeloid leukemia (LAML) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"laml_mut_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://www.synapse.org/#!Synapse:syn1729383"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">196</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript mutationMatrix processed on 2015-01-28"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA PANCAN strictly filtered maf files (file names: *_cleaned_filtered.maf) download from Synapse, processed into gene by sample matrix at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA acute myeloid leukemia (LAML) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"laml_mut01_ucsc_mutation"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">"https://www.synapse.org/#!Synapse:syn1729383"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">196</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript mutationMatrix processed on 2015-01-28"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA PANCAN strictly filtered maf files (file names: *_cleaned_filtered.maf) download from Synapse, processed into gene by sample matrix at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA acute myeloid leukemia (LAML) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>laml_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>191</td>
<td>TCGA acute myeloid leukemia (LAML) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
<tr>
<td>laml_mut_ucsc_mutation</td>
<td>ucsc</td>
<td>mut</td>
<td>196</td>
<td>TCGA acute myeloid leukemia (LAML) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
<tr>
<td>laml_mut01_ucsc_mutation</td>
<td>ucsc</td>
<td>mut01</td>
<td>196</td>
<td>TCGA acute myeloid leukemia (LAML) somatic mutation data. Red (=1) indicates that a non-silent somatic mutation (nonsense, missense, frame-shif indels, splice site mutations, stop codon readthroughs) was identified in the protein coding region of a gene, or any mutation identified in a non-coding gene. White (=0) indicates that none of the above mutation calls were made in this gene for the specific sample.</td>
</tr>
</tbody></table>

<h2 id="sarc-sarcoma">SARC - Sarcoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>sarc_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_followup-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_newtumor-followup-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>sarc_cnv_ucsc_gistic2thd</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"sarc_cnv_ucsc_gistic2thd"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" http://gdac.broadinstitute.org/runs/analyses__2014_10_17/data/SARC/20141017/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">257</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"   cgData TCGAscript CopyNumber_Gistic2 processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"FIREHOSE data download from TCGA DCC, processed at UCSC into cgData repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA sarcoma (SARC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method."</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>sarc_cnv_ucsc_gistic2thd</td>
<td>ucsc</td>
<td>cnv</td>
<td>257</td>
<td>TCGA sarcoma (SARC) thresholded gene-level copy number variation (CNV) estimated using the GISTIC2 method.</td>
</tr>
</tbody></table>

<h2 id="stad-stomach-adenocarcinoma">STAD - Stomach adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>stad_mut_ucsc_mutationbcmgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut</td>
</tr>
<tr>
<td>stad_mut01_ucsc_mutationbcmgene</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"stad_mut_ucsc_mutationbcmgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/stad/gsc/hgsc.bcm.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">379</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA stomach adenocarcinoma (STAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"stad_mut01_ucsc_mutationbcmgene"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">" https://tcga-data.nci.nih.gov/tcgafiles/ftp_auth/distro_ftpusers/anonymous/tumor/stad/gsc/hgsc.bcm.edu/illuminaga_dnaseq_automated/mutations/"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">379</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cgData TCGAscript maf processed on 2015-01-27"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">"Download .maf file from TCGA DCC, processed into gene by sample matrix at UCSC, stored in the UCSC Xena repository"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">"TCGA stomach adenocarcinoma (STAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method. "</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>stad_mut_ucsc_mutationbcmgene</td>
<td>ucsc</td>
<td>mut</td>
<td>379</td>
<td>TCGA stomach adenocarcinoma (STAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method.</td>
</tr>
<tr>
<td>stad_mut01_ucsc_mutationbcmgene</td>
<td>ucsc</td>
<td>mut01</td>
<td>379</td>
<td>TCGA stomach adenocarcinoma (STAD) somatic mutation data. Sequencing data are generated on a IlluminaGA system. The calls are generated at Baylor College of Medicine Human Genome Sequencing Center using the Baylor pipeline method.</td>
</tr>
</tbody></table>

<h2 id="read-rectum-adenocarcinoma">READ - Rectum adenocarcinoma</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>read_patient_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_drug_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_radiation_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_followup-v1p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_newtumor_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
<tr>
<td>read_othermalignancy-v4p0_tcga_na</td>
<td>clinical</td>
<td>TCGA</td>
<td></td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>

<h2 id="brain-lower-grade-glioma-amp-glioblastoma-multiforme">BRAIN - Lower Grade Glioma &amp; Glioblastoma multiforme</h2>

<table><thead>
<tr>
<th>Collection Name</th>
<th>Collection Type</th>
<th>Data Source</th>
<th>Data Type</th>
</tr>
</thead><tbody>
<tr>
<td>brain_color_tcga_import</td>
<td>category</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_patient_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_followup-v1p0_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_drug_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_newtumor_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_othermalignancy-v4p0_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_events_tcga_clinical</td>
<td>clinical</td>
<td>tcga</td>
<td>color</td>
</tr>
<tr>
<td>brain_cnv_ucsc_gistic</td>
<td>molecular</td>
<td>ucsc</td>
<td>cnv</td>
</tr>
<tr>
<td>brain_mut01_ucsc_import</td>
<td>molecular</td>
<td>ucsc</td>
<td>mut01</td>
</tr>
<tr>
<td>brain_cnv_cbio_gistic</td>
<td>molecular</td>
<td>cBio</td>
<td>cnv</td>
</tr>
<tr>
<td>brain_mut_cbio_wxs</td>
<td>molecular</td>
<td>cBio</td>
<td>mut</td>
</tr>
<tr>
<td>brain_mut01_cbio_import</td>
<td>molecular</td>
<td>cBio</td>
<td>mut01</td>
</tr>
<tr>
<td>brain_methylation_cbio_hm27</td>
<td>molecular</td>
<td>cBio</td>
<td>methylation</td>
</tr>
<tr>
<td>brain_rna_cbio_rnaseq-bc</td>
<td>molecular</td>
<td>cBio</td>
<td>rna</td>
</tr>
<tr>
<td>brain_protein_cbio_rppa-zscore</td>
<td>molecular</td>
<td>cBio</td>
<td>protein</td>
</tr>
</tbody></table>

<h3 id="more-details-of-molecular-collections">More Details of Molecular Collections</h3>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"cnv"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brain_cnv_ucsc_gistic"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">1090</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>
<pre class="highlight json"><code><span class="w">
</span><span class="p">{</span><span class="w">
    </span><span class="nt">"source"</span><span class="p">:</span><span class="w"> </span><span class="s2">"ucsc"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"type"</span><span class="p">:</span><span class="w"> </span><span class="s2">"mut01"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"collection"</span><span class="p">:</span><span class="w"> </span><span class="s2">"brain_mut01_ucsc_import"</span><span class="p">,</span><span class="w">
    </span><span class="nt">"RawDataUrl"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"sampleSize"</span><span class="p">:</span><span class="w"> </span><span class="mi">818</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wrangler"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"wranglingProcedure"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="p">,</span><span class="w">
    </span><span class="nt">"description"</span><span class="p">:</span><span class="w"> </span><span class="s2">""</span><span class="w">
</span><span class="p">}</span><span class="w">
</span></code></pre>

<table><thead>
<tr>
<th>Collection</th>
<th>Data Source</th>
<th>Data Type</th>
<th>Size</th>
<th>Description</th>
</tr>
</thead><tbody>
<tr>
<td>brain_cnv_ucsc_gistic</td>
<td>ucsc</td>
<td>cnv</td>
<td>1090</td>
<td></td>
</tr>
<tr>
<td>brain_mut01_ucsc_import</td>
<td>ucsc</td>
<td>mut01</td>
<td>818</td>
<td></td>
</tr>
</tbody></table>

          <h1 id="tools">Tools</h1>

<h3 id="markers-patients">Markers + Patients</h3>

<p>Link copy number variation and mutation data to patients.</p>

<h3 id="timelines">Timelines</h3>

<h3 id="survival">Survival</h3>

<p>Compare survival rates of selected patients against the remaining population in a Kaplan Meier plot.</p>

<h3 id="pathways">Pathways</h3>

<p>Map patient specific expression levels on a hand curated network of genes.</p>

<h3 id="patient-data">Patient Data</h3>

<h3 id="plsr">PLSR</h3>

<p>Use linear regression to correlate genes with clinical features using RNA expression.</p>

<h3 id="pca">PCA</h3>

<p>Two dimensional view of per sample expression data.</p>

<h3 id="oncoprint">Oncoprint</h3>

<h3 id="geneset-test">Geneset Test</h3>

<h3 id="data-explorer">Data Explorer</h3>

<h3 id="metadata">MetaData</h3>

      </div>
      <div class="dark-box">
          <div class="lang-selector">
                <a href="#" data-language-name="python">python</a>
                <a href="#" data-language-name="javascript">javascript</a>
                <a href="#" data-language-name="r">r</a>
          </div>
      </div>
    </div>
  </body>
</html>
